<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Wowchemy 5.2.0 for Hugo"><meta name=description content="Maxime Borry, PhD."><link rel=alternate hreflang=en-us href=https://maximeborry.com/><link rel=preconnect href=https://fonts.gstatic.com crossorigin><meta name=theme-color content="#01579b"><script src=/js/mathjax-config.js></script><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css integrity="sha256-FMvZuGapsJLjouA6k7Eo2lusoAX9i0ShlWFG6qt7SLc=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin=anonymous media=print onload="this.media='all'"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/github.min.css crossorigin=anonymous title=hl-light media=print onload="this.media='all'"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/github.min.css crossorigin=anonymous title=hl-dark media=print onload="this.media='all'" disabled><script src=https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js integrity crossorigin=anonymous async></script><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Bebas+Neue%7CIBM+Plex+Mono&display=swap"><link rel=stylesheet href=/css/wowchemy.a2cc92da6d94187535714538e195ecb0.css><script async src="https://www.googletagmanager.com/gtag/js?id=UA-77504401-1"></script><script>window.dataLayer=window.dataLayer||[];function gtag(){dataLayer.push(arguments);}
function trackOutboundLink(url,target){gtag('event','click',{'event_category':'outbound','event_label':url,'transport_type':'beacon','event_callback':function(){if(target!=='_blank'){document.location=url;}}});console.debug("Outbound link clicked: "+url);}
function onClickCallback(event){if((event.target.tagName!=='A')||(event.target.host===window.location.host)){return;}
trackOutboundLink(event.target,event.target.getAttribute('target'));}
gtag('js',new Date());gtag('config','UA-77504401-1',{'anonymize_ip':true});gtag('set',{'cookie_flags':'SameSite=None;Secure'});document.addEventListener('click',onClickCallback,false);</script><link rel=alternate href=/index.xml type=application/rss+xml title="Maxime Borry"><link rel=manifest href=/index.webmanifest><link rel=icon type=image/png href=/media/icon_hu4c4b160219548583b0b35e67c5987186_165157_32x32_fill_lanczos_center_2.png><link rel=apple-touch-icon type=image/png href=/media/icon_hu4c4b160219548583b0b35e67c5987186_165157_180x180_fill_lanczos_center_2.png><link rel=canonical href=https://maximeborry.com/><meta property="twitter:card" content="summary"><meta property="twitter:site" content="@notmaxib"><meta property="twitter:creator" content="@notmaxib"><meta property="og:site_name" content="Maxime Borry"><meta property="og:url" content="https://maximeborry.com/"><meta property="og:title" content="Maxime Borry"><meta property="og:description" content="Maxime Borry, PhD."><meta property="og:image" content="https://maximeborry.com/media/icon_hu4c4b160219548583b0b35e67c5987186_165157_512x512_fill_lanczos_center_2.png"><meta property="twitter:image" content="https://maximeborry.com/media/icon_hu4c4b160219548583b0b35e67c5987186_165157_512x512_fill_lanczos_center_2.png"><meta property="og:locale" content="en-us"><meta property="og:updated_time" content="2023-08-02T00:00:00+00:00"><script type=application/ld+json>{"@context":"https://schema.org","@type":"WebSite","url":"https://maximeborry.com/"}</script><script src=https://cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.1.1/cookieconsent.min.js integrity="sha256-5VhCqFam2Cn+yjw61zbBNrbHVJ6SRydPeKopYlngbiQ=" crossorigin=anonymous></script><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.1.1/cookieconsent.min.css integrity="sha256-zQ0LblD/Af8vOppw18+2anxsuaz3pWYyVWi+bTvTH8Q=" crossorigin=anonymous><script>window.addEventListener("load",function(){window.cookieconsent.initialise({"palette":{"popup":{"background":"#01579b","text":"rgb(255, 255, 255)"},"button":{"background":"rgb(255, 255, 255)","text":"#01579b"}},"theme":"classic","content":{"message":"This website uses cookies to ensure you get the best experience on our website.","dismiss":"Got it!","link":"Learn more","href":"https://www.cookiesandyou.com"}})});</script><title>Maxime Borry</title></head><body id=top data-spy=scroll data-offset=70 data-target=#navbar-main class=page-wrapper><script src=/js/wowchemy-init.min.5f3407a3a06f15b985245df562932996.js></script><div class=page-header><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container-xl><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=/>Maxime Borry</a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar-content aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=/>Maxime Borry</a></div><div class="navbar-collapse main-menu-item collapse justify-content-end" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=/#hero data-target=#hero><span>Home</span></a></li><li class=nav-item><a class=nav-link href=/#publications data-target=#publications><span>Publications</span></a></li><li class=nav-item><a class=nav-link href=/courses><span>Courses</span></a></li><li class=nav-item><a class=nav-link href=/#posts data-target=#posts><span>Blog</span></a></li><li class=nav-item><a class=nav-link href=/#contact data-target=#contact><span>Contact me</span></a></li><li class=nav-item><a class=nav-link href=/maxime_borry_cv.pdf><span>CV</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class="nav-item dropdown theme-dropdown"><a href=# class=nav-link data-toggle=dropdown aria-haspopup=true aria-label="Display preferences"><i class="fas fa-moon" aria-hidden=true></i></a><div class=dropdown-menu><a href=# class="dropdown-item js-set-theme-light"><span>Light</span></a>
<a href=# class="dropdown-item js-set-theme-dark"><span>Dark</span></a>
<a href=# class="dropdown-item js-set-theme-auto"><span>Automatic</span></a></div></li></ul></div></nav></div><div class=page-body><span class="js-widget-page d-none"></span><section id=hero class="home-section wg-hero dark"><div class="home-section-bg bg-image parallax" style=background-image:url(https://maximeborry.com/media/climbing.jpg);background-size:cover;background-position:50%;filter:brightness(0.3)></div><div class=container><h1 class=hero-title>Maxime Borry, PhD.<br>Postdoctoral Researcher</h1><div class=hero-lead><p>Working on <strong>Bioinformatics</strong> for <strong>ancient DNA Metagenomics</strong><br>at the<br><strong>Max Planck Institute for Evolutionary Anthropology</strong></p></div></div></section><section id=about class="home-section wg-about"><div class=home-section-bg></div><div class=container><div class=row><div class="col-12 col-lg-4"><div id=profile><img class="avatar avatar-circle" src=/authors/maxime/avatar_hu55fedc832821a50ac8f2b09fe94bc985_478322_270x270_fill_lanczos_center_2.png alt><div class=portrait-title><h2></h2><h3>Maxime Borry, PhD.</h3><h3><a href=https://www.eva.mpg.de/index/ target=_blank rel=noopener><span>Max Planck Institute for Evolutionary Anthropology</span></a></h3><h3><a href=https://www.leibniz-hki.de/en/ target=_blank rel=noopener><span>Leibniz Institut for Natural Product Research and Infection Biology Hans KnÃ¶ll Institute</span></a></h3></div><ul class=network-icon aria-hidden=true><li><a href=https://github.com/maxibor target=_blank rel=noopener aria-label=github><i class="fab fa-github big-icon"></i></a></li><li><a href=https://linkedin.com/in/maximeborry/ target=_blank rel=noopener aria-label=linkedin><i class="fab fa-linkedin big-icon"></i></a></li><li><a href=/maxime_borry_cv.pdf aria-label=cv><i class="ai ai-cv big-icon"></i></a></li><li><a href=/#contact aria-label=envelope><i class="fas fa-envelope big-icon"></i></a></li><li><a href=https://twitter.com/notmaxib target=_blank rel=noopener aria-label=twitter><i class="fab fa-twitter big-icon"></i></a></li></ul></div></div><div class="col-12 col-lg-8"><h1>Hey there,</h1><div class=article-style><p>I am <strong>Maxime</strong>, a <strong>Postdoctoral Researcher in Bioinformatics</strong> at the <a href=https://www.leibniz-hki.de/en/ target=_blank rel=noopener><strong>Leibniz Institut for Natural Product Research and Infection Biology Hans KnÃ¶ll Institute</strong></a>, and the <a href=https://www.eva.mpg.de/index/ target=_blank rel=noopener><strong>Max Planck Institute for Evolutionary Anthropology</strong></a> in Germany.</p><p>After an undergraduate degree in biology, I first decided to dive in the studies of population and our environment. Only after realizing I wanted to focus more on the statistical part and the data analysis, I switched back to a Bioinformatics MSc.</p><p>I did my PhD in the <a href=https://www.shh.mpg.de/118825/dag_research_groups target=_blank rel=noopener>Microbiome Sciences research group</a> on ancient DNA metagenomics, under the supervision of <a href=http://christinawarinner.com target=_blank rel=noopener>Christina Warinner</a>. I now continue this work as a postdoctoral researcher, focusing more in depth on fermentation microbiomes.</p></div><div class=row><div class=col-md-5><div class=section-subheading>Interests</div><ul class="ul-interests mb-0"><li>Bioinformatics</li><li>Metagenomics</li><li>Machine Learning</li></ul></div><div class=col-md-7><div class=section-subheading>Education</div><ul class="ul-edu fa-ul mb-0"><li><i class="fa-li fas fa-graduation-cap"></i><div class=description><p class=course>PhD candidate in Bioinformatics, 2018-2023</p><p class=institution>Friedrich-Schiller-UniversitÃ¤t Jena ðŸ‡©ðŸ‡ª</p></div></li><li><i class="fa-li fas fa-graduation-cap"></i><div class=description><p class=course>M.Sc. in Bioinformatics, 2016-2018</p><p class=institution>UniversitÃ© Paris Diderot ðŸ‡«ðŸ‡·</p></div></li><li><i class="fa-li fas fa-graduation-cap"></i><div class=description><p class=course>M.Sc. in Ecology and Environmental sciences, 2014-2015</p><p class=institution>UniversitÃ© Joseph Fourier, Grenoble ðŸ‡«ðŸ‡·</p></div></li><li><i class="fa-li fas fa-graduation-cap"></i><div class=description><p class=course>Exchange Semester, 2013</p><p class=institution>University of Nordland ðŸ‡³ðŸ‡´</p></div></li><li><i class="fa-li fas fa-graduation-cap"></i><div class=description><p class=course>B.Sc. in Life Sciences, 2011-2014</p><p class=institution>UniversitÃ© Paris Descartes ðŸ‡«ðŸ‡·</p></div></li></ul></div></div></div></div></div></section><section id=publications class="home-section wg-pages"><div class=home-section-bg></div><div class=container><div class=row><div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start"><h1 class=mb-0>Recent Publications</h1></div><div class="col-12 col-lg-8"><div class="alert alert-note"><div>Quickly discover relevant content by <a href=./publication/>filtering publications</a>.</div></div><div class=pub-list-item style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i><span class="article-metadata li-cite-author"><span>M Borry</span>, <span>A Forsythe</span>, <span>A Andrades ValtueÃ±a</span>, <span>A HÃ¼bner</span>, <span>A Ibrahim</span>, <span>A Quagliariello</span>, <span>AE White</span>, <span>A Kocher</span>, <span>Ã…J VÃ¥gene</span>, <span>BP Bartholdy</span>, <span>D SpurÄ«te</span>, <span>GY Ponce-Soto</span>, <span>G Neumann</span>, <span>IT Huang</span>, <span>I Light</span>, <span>IM Velsko</span>, <span>I Jackson</span>, <span>J Frangenberg</span>, <span>JG Serrano</span>, <span>J Fumey</span>, <span>KT Ã–zdoÄŸan</span>, <span>KE Blevins</span>, <span>KG Daly</span>, <span>M Lopopolo</span>, <span>M Moraitou</span>, <span>M Michel</span>, <span>M van Os</span>, <span>MJ Bravo-Lopez</span>, <span>MS Sarhan</span>, <span>ND Dagtas</span>, <span>N Oskolkov</span>, <span>OS Smith</span>, <span>O Lebrasseur</span>, <span>P Rozwalak</span>, <span>R Eisenhofer</span>, <span>S Wasef</span>, <span>SL Ramachandran</span>, <span>V Vanghi</span>, <span>C Warinner</span>, <span>JA Fellows Yates</span>.</span>
<a href=/publication/borry-2023-amdirt/>Facilitating accessible, rapid, and appropriate processing of ancient metagenomic data with AMDirT</a>.
<em>F1000Research</em>,
2023.<div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href="https://f1000research.com/articles/12-926/v1/pdf?article_uuid=4c0938e5-e099-474f-953a-292a08a830fe" target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/borry-2023-amdirt/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/SPAAM-community/AMDirT target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.12688/f1000research.134798.1 target=_blank rel=noopener>DOI</a></div></div><div class=pub-list-item style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i><span class="article-metadata li-cite-author"><span>Moritz E. Beber</span>, <span>Maxime Borry</span>, <span>Sofia Stamouli</span>, <span>James A. Fellows Yates</span>.</span>
<a href=/publication/beber-2023/>TAXPASTA: TAXonomic Profile Aggregation and STAndardisation</a>.
<em>Journal of Open Source Software</em>,
2023.<div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.theoj.org/joss-papers/joss.05627/10.21105.joss.05627.pdf target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/beber-2023/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/taxprofiler/taxpasta target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.21105/joss.05627 target=_blank rel=noopener>DOI</a></div></div><div class=pub-list-item style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i><span class="article-metadata li-cite-author"><span>Martin Klapper</span>, <span>Alexander HÃ¼bner</span>, <span>Anan Ibrahim</span>, <span>Ina Wasmuth</span>, <span>Maxime Borry</span>, <span>Veit G. Haensch</span>, <span>Shuaibing Zhang</span>, <span>Walid K. Al-Jammal</span>, <span>Harikumar Suma</span>, <span>James A. Fellows Yates</span>, <span>Jasmin Frangenberg</span>, <span>Irina M. Velsko</span>, <span>Somak Chowdhury</span>, <span>Rosa Herbst</span>, <span>Evgeni V. Bratovanov</span>, <span>Hans-Martin Dahse</span>, <span>Therese Horch</span>, <span>Christian Hertweck</span>, <span>Manuel Ramon GonzÃ¡lez Morales</span>, <span>Lawrence Guy Straus</span>, <span>Ivan Vilotijevic</span>, <span>Christina Warinner</span>, <span>Pierre Stallforth</span>.</span>
<a href=/publication/klapper-natural-products-reconstructed-2023/>Natural Products from Reconstructed Bacterial Genomes of the Middle and Upper Paleolithic</a>.
<em>Science</em>,
2023.<div class=btn-links><a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/klapper-natural-products-reconstructed-2023/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.1126/science.adf5300 target=_blank rel=noopener>DOI</a></div></div><div class=pub-list-item style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i><span class="article-metadata li-cite-author"><span>Maxime Borry</span>, <span>Alexander HÃ¼bner</span>, <span>Christina Warinner</span>.</span>
<a href=/publication/borry-2022/>sam2lca: Lowest Common Ancestor for SAM/BAM/CRAM alignment files</a>.
<em>Journal of Open Source Software</em>,
2022.<div class=btn-links><a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/borry-2022/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.21105/joss.04360 target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/maxibor/sam2lca target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://joss.theoj.org/papers/10.21105/joss.04360.pdf target=_blank rel=noopener>PDF</a></div></div><div class=pub-list-item style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i><span class="article-metadata li-cite-author"><span>Maxime Borry</span>, <span>Alexander HÃ¼bner</span>, <span>Adam B. Rohrlach</span>, <span>Christina Warinner</span>.</span>
<a href=/publication/borry-pydamage-2021/>PyDamage: automated ancient damage identification and estimation for contigs in ancient DNA de novo assembly</a>.
<em>PeerJ</em>,
2021.<div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://peerj.com/articles/11845.pdf target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/borry-pydamage-2021/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/maxibor/pydamage target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.7717/peerj.11845 target=_blank rel=noopener>DOI</a></div></div><div class=pub-list-item style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i><span class="article-metadata li-cite-author"><span>Marsha C. Wibowo</span>, <span>Zhen Yang</span>, <span>Maxime Borry</span>, <span>Alexander HÃ¼bner</span>, <span>Kun D. Huang</span>, <span>Braden T. Tierney</span>, <span>Samuel Zimmerman</span>, <span>Francisco Barajas-Olmos</span>, <span>Cecilia Contreras-Cubas</span>, <span>Humberto GarcÃ­a-Ortiz</span>, <span>AngÃ©lica MartÃ­nez-HernÃ¡ndez</span>, <span>Jacob M. Luber</span>, <span>Philipp Kirstahler</span>, <span>Tre Blohm</span>, <span>Francis E. Smiley</span>, <span>Richard Arnold</span>, <span>Sonia A. Ballal</span>, <span>SÃ¼nje Johanna Pamp</span>, <span>Julia Russ</span>, <span>Frank Maixner</span>, <span>Omar Rota-Stabelli</span>, <span>Nicola Segata</span>, <span>Karl Reinhard</span>, <span>Lorena Orozco</span>, <span>Christina Warinner</span>, <span>Meradeth Snow</span>, <span>Steven LeBlanc</span>, <span>Aleksandar D. Kostic</span>.</span>
<a href=/publication/wibowo-reconstruction-2021/>Reconstruction of ancient microbial genomes from the human gut</a>.
<em>Nature</em>,
2021.<div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.nature.com/articles/s41586-021-03532-0.pdf target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/wibowo-reconstruction-2021/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.1038/s41586-021-03532-0 target=_blank rel=noopener>DOI</a></div></div><div class=pub-list-item style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i><span class="article-metadata li-cite-author"><span>James A. Fellows Yates</span>, <span>Thiseas C. Lamnidis</span>, <span>Maxime Borry</span>, <span>Aida Andrades ValtueÃ±a</span>, <span>Zandra FagernÃ¤s</span>, <span>Stephen Clayton</span>, <span>Maxime U. Garcia</span>, <span>Judith Neukamm</span>, <span>Alexander Peltzer</span>.</span>
<a href=/publication/yates-reproducible-2021/>Reproducible, portable, and efficient ancient genome reconstruction with nf-core/eager</a>.
<em>PeerJ</em>,
2021.<div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://peerj.com/articles/10947 target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/yates-reproducible-2021/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.7717/peerj.10947 target=_blank rel=noopener>DOI</a></div></div><div class=pub-list-item style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i><span class="article-metadata li-cite-author"><span>James A. Fellows Yates</span>, <span>Aida Andrades ValtueÃ±a</span>, <span>Ã…shild J. VÃ¥gene</span>, <span>Becky Cribdon</span>, <span>Irina M. Velsko</span>, <span>Maxime Borry</span>, <span>Miriam J. Bravo-Lopez</span>, <span>Antonio Fernandez-Guerra</span>, <span>Eleanor J. Green</span>, <span>Shreya L. Ramachandran</span>, <span>Peter D. Heintzman</span>, <span>Maria A. Spyrou</span>, <span>Alexander HÃ¼bner</span>, <span>Abigail S. Gancz</span>, <span>Jessica Hider</span>, <span>Aurora F. Allshouse</span>, <span>Valentina Zaro</span>, <span>Christina Warinner</span>.</span>
<a href=/publication/fellows-yates-community-curated-2021/>Community-curated and standardised metadata of published ancient metagenomic samples with AncientMetagenomeDir</a>.
<em>Scientific Data</em>,
2021.<div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.nature.com/articles/s41597-021-00816-y.pdf target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/fellows-yates-community-curated-2021/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.1038/s41597-021-00816-y target=_blank rel=noopener>DOI</a></div></div><div class=pub-list-item style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i><span class="article-metadata li-cite-author"><span>Maxime Borry et al</span>.</span>
<a href=/publication/borry-coproid-2020/>CoproID predicts the source of coprolites and paleofeces using microbiome composition and host DNA content</a>.
<em>PeerJ</em>,
2020.<div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://peerj.com/articles/9001 target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/borry-coproid-2020/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/nf-core/coproid target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.7717/peerj.9001 target=_blank rel=noopener>DOI</a></div></div><div class=pub-list-item style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i><span class="article-metadata li-cite-author"><span>Maxime Borry</span>.</span>
<a href=/publication/borry-sourcepredict-2019/>Sourcepredict: Prediction of metagenomic sample sources using dimension reduction followed by machine learning classification</a>.
<em>Journal of Open Source Software</em>,
2019.<div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://joss.theoj.org/papers/10.21105/joss.01540 target=_blank rel=noopener>PDF</a>
<a href=# class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal" data-filename=/publication/borry-sourcepredict-2019/cite.bib>Cite</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/maxibor/sourcepredict target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.21105/joss.01540 target=_blank rel=noopener>DOI</a></div></div></div></div></div></section><section id=experience class="home-section wg-experience"><div class=home-section-bg></div><div class=container><div class=row><div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start"><h1 class=mb-0>Experience</h1></div><div class="col-12 col-lg-8"><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class=col>&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border exp-fill">&nbsp;</span></div><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="section-subheading card-title exp-title text-muted my-0">Postdoctoral Researcher in Bioinformatics</div><div class="section-subheading card-title exp-company text-muted my-0"><a href=https://www.leibniz-hki.de/en/ target=_blank rel=noopener>Leibniz Institut for Natural Product Research and Infection Biology Hans KnÃ¶ll Institute</a></div><div class="text-muted exp-meta">Apr 2023 â€“
Present
<span class=middot-divider></span><span>Leipzig, Germany</span></div><div class=card-text>Bioinformatic metagenomic analysis of ancient fermentation microbiomes.</div></div></div></div></div><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border">&nbsp;</span></div><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="section-subheading card-title exp-title text-muted my-0">Doctoral Researcher in Bioinformatics</div><div class="section-subheading card-title exp-company text-muted my-0"><a href=https://www.eva.mpg.de target=_blank rel=noopener>Max Planck Institute for Evolutionary Anthropology</a></div><div class="text-muted exp-meta">Sep 2021 â€“
Mar 2023
<span class=middot-divider></span><span>Leipzig, Germany</span></div><div class=card-text>Development of tools and pipelines for the analysis of ancient DNA metagenomics data.</div></div></div></div></div><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border exp-fill">&nbsp;</span></div><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="section-subheading card-title exp-title text-muted my-0">Bioinformatics Doctoral Researcher</div><div class="section-subheading card-title exp-company text-muted my-0"><a href=https://www.shh.mpg.de/ target=_blank rel=noopener>Max Planck Institute for the Science of Human History</a></div><div class="text-muted exp-meta">Sep 2018 â€“
Present
<span class=middot-divider></span><span>Jena, Germany</span></div><div class=card-text>Development of tools and pipelines for the analysis of ancient DNA metagenomics data.</div></div></div></div></div><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border">&nbsp;</span></div><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="section-subheading card-title exp-title text-muted my-0">Bioinformatics Research intern</div><div class="section-subheading card-title exp-company text-muted my-0"><a href="http://ecoanthropologie.cnrs.fr/article833.html?lang=en" target=_blank rel=noopener>Natural History Museum - CNRS</a></div><div class="text-muted exp-meta">Jan 2018 â€“
Jul 2018
<span class=middot-divider></span><span>Paris, France</span></div><div class=card-text>Development of tools and pipelines for the analysis of ancient DNA metagenomics data.</div></div></div></div></div><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border">&nbsp;</span></div><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="section-subheading card-title exp-title text-muted my-0">Bioinformatics Research intern</div><div class="section-subheading card-title exp-company text-muted my-0"><a href=https://research.pasteur.fr/fr/team/bioinformatics-and-biostatistics-hub/ target=_blank rel=noopener>Institut Pasteur</a></div><div class="text-muted exp-meta">Mar 2017 â€“
May 2017
<span class=middot-divider></span><span>Paris, France</span></div><div class=card-text>Development of a tool in R and Shiny for comparison of expression level in multi-omics data.</div></div></div></div></div><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border">&nbsp;</span></div><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="section-subheading card-title exp-title text-muted my-0">Bioinformatics Research assistant</div><div class="section-subheading card-title exp-company text-muted my-0"><a href=https://www.crg.eu/toni_gabaldon target=_blank rel=noopener>CRG</a></div><div class="text-muted exp-meta">Jun 2016 â€“
Aug 2016
<span class=middot-divider></span><span>Barcelona, Spain</span></div><div class=card-text>Exploration of the human oral mycobiome.</div></div></div></div></div><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border">&nbsp;</span></div><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="section-subheading card-title exp-title text-muted my-0">Research intern</div><div class="section-subheading card-title exp-company text-muted my-0"><a href=https://www.eva.mpg.de/index.html target=_blank rel=noopener>MPI-EVA</a></div><div class="text-muted exp-meta">Dec 2015 â€“
Feb 2016
<span class=middot-divider></span><span>Leipzig, Germany</span></div><div class=card-text>Detection of gene duplications between Neanderthal and Mordern Humans.</div></div></div></div></div><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border">&nbsp;</span></div><div class="row h-50"><div class=col>&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="section-subheading card-title exp-title text-muted my-0">Guest researcher</div><div class="section-subheading card-title exp-company text-muted my-0"><a href=https://www.nhm.uio.no/english/research/groups/pet/ target=_blank rel=noopener>University of Oslo</a></div><div class="text-muted exp-meta">Jun 2015 â€“
Oct 2015
<span class=middot-divider></span><span>Oslo, Norway</span></div><div class=card-text>Development of new markers and metagenomics softwares for plant identification. De novo and mapping assembly of Illumina data, phylogeny of the Anacyclus genus, and metabarcoding visualization software development.</div></div></div></div></div></div></div></div></section><section id=accomplishments class="home-section wg-accomplishments"><div class=home-section-bg></div><div class=container><div class=row><div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start"><h1 class=mb-0>AccomplishÂ­ments</h1></div><div class="col-12 col-lg-8"><div class="card experience course"><div class=card-body><a href=https://www.isbarch.org/poster_prize target=_blank rel=noopener><div class="section-subheading card-title exp-title text-muted my-0">Future Fellows Poster Prize at ISBA 10 conference</div></a><div class="card-subtitle my-0 article-metadata"><a href=https://www.isbarch.org target=_blank rel=noopener>International Society for Biomolecular Archaeology (ISBA)</a><span class=middot-divider></span>
Sep 2023</div></div></div><div class="card experience course"><div class=card-body><div class="section-subheading card-title exp-title text-muted my-0">Postdoctoral researcher grant</div><div class="card-subtitle my-0 article-metadata"><a href=https://www.microverse-cluster.de/en/ target=_blank rel=noopener>Balance of the Microverse Cluster of Excellence</a><span class=middot-divider></span>
Mar 2023</div></div></div><div class="card experience course"><div class=card-body><a href=https://www.sapweb.fr/index.php/la-societe/archive-des-prix.html target=_blank rel=noopener><div class="section-subheading card-title exp-title text-muted my-0">Best poster award for &ldquo;Etude du microbiote ancien Ã  partir de l&rsquo;ADN ancien appliquÃ© Ã  la civilisation de l&rsquo;Oxus Ã  l&rsquo;Ã¢ge du Bronze&rdquo;</div></a><div class="card-subtitle my-0 article-metadata"><a href=https://www.sapweb.fr target=_blank rel=noopener>Paris Anthropological Society</a><span class=middot-divider></span>
Jan 2019</div></div></div><div class="card experience course"><div class=card-body><a href=https://www.meet-u.org/edition_2018.html target=_blank rel=noopener><div class="section-subheading card-title exp-title text-muted my-0">Winning team</div></a><div class="card-subtitle my-0 article-metadata"><a href=https://www.meet-u.org target=_blank rel=noopener>Meet-U</a><span class=middot-divider></span>
Mar 2018</div><div class=card-text>Winning team of the Bioinformatics competition on protein-protein docking.</div></div></div></div></div></div></section><section id=posts class="home-section wg-pages"><div class=home-section-bg></div><div class=container><div class=row><div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start"><h1 class=mb-0>Recent Posts</h1></div><div class="col-12 col-lg-8"><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/post/monty-hall-python/>Using Python to solve the Monty Hall problem</a></div><a href=/post/monty-hall-python/ class=summary-link><div class=article-style>What is the best strategy to win a ðŸš— ? Keeping the original choice, or changing of door ?</div></a><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>May 13, 2022</span>
<span class=middot-divider></span><span class=article-reading-time>3 min read</span></div></div></div><div class=ml-3><a href=/post/monty-hall-python/><img src=/post/monty-hall-python/featured_hu3fe5ba728fae0a091e9d27528c225c0f_28102_150x0_resize_lanczos_2.png alt="Using Python to solve the Monty Hall problem" loading=lazy></a></div></div><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/post/kraken-uniq/>A new E-score for KrakenUniq</a></div><a href=/post/kraken-uniq/ class=summary-link><div class=article-style>How to distinguish between true and false positive assignations with KrakenUniq</div></a><div class="stream-meta article-metadata"><div class=article-metadata><div><span></span></div><span class=article-date>May 10, 2022</span>
<span class=middot-divider></span><span class=article-reading-time>7 min read</span></div></div></div><div class=ml-3><a href=/post/kraken-uniq/><img src=/post/kraken-uniq/featured_hu6bdd134084c2c2ea375ebf18fabcb0c5_21973_150x0_resize_lanczos_2.png alt="A new E-score for KrakenUniq" loading=lazy></a></div></div><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/post/rocksdb/>noSQL: when it's too big for a dictionary</a></div><a href=/post/rocksdb/ class=summary-link><div class=article-style>Recently, I&rsquo;ve been working on a project where I need to store a key-value pairs for very large files. In Python, the first idea would be to use a dictionary.</div></a><div class="stream-meta article-metadata"><div class=article-metadata><div><span></span></div><span class=article-date>Last updated on
Feb 10, 2021</span>
<span class=middot-divider></span><span class=article-reading-time>3 min read</span></div></div></div><div class=ml-3><a href=/post/rocksdb/><img src=/post/rocksdb/featured_hueb0ca10c5d2551f34f4b9bf650eedc76_65958_150x0_resize_q75_lanczos.jpg alt="noSQL: when it's too big for a dictionary" loading=lazy></a></div></div><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/post/interactive-map-howto/>Making Interactive maps in Python using GeoJSON and GitHub</a></div><a href=/post/interactive-map-howto/ class=summary-link><div class=article-style>Recently, I&rsquo;ve been involved with the AncientMetagenomeDir project. Briefly, with this collaborative community effort, we aimed to regroup in one single repository, all the metadata about every single published ancient DNA metagenomics article, and turn them into FAIR scientific data.</div></a><div class="stream-meta article-metadata"><div class=article-metadata><span class=article-date>Jan 12, 2021</span>
<span class=middot-divider></span><span class=article-reading-time>3 min read</span></div></div></div><div class=ml-3><a href=/post/interactive-map-howto/><img src=/post/interactive-map-howto/featured_hu735c323bb12acc9c85b3da083adb53a6_434394_150x0_resize_lanczos_2.png alt="Making Interactive maps in Python using GeoJSON and GitHub" loading=lazy></a></div></div></div></div></div></section><section id=projects class="home-section wg-portfolio"><div class=home-section-bg></div><div class=container><div class=row><div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start"><h1 class=mb-0>A few other projects I&rsquo;ve done</h1><p class=mt-1>(more on <a href=https://github.com/maxibor>GitHub</a>)</p></div><div class="col-12 col-lg-8"><span class="d-none default-project-filter">*</span><div class=project-toolbar><div class=project-filters><div class=btn-toolbar><div class="btn-group flex-wrap"><a href=# data-filter=* class="btn btn-primary btn-lg active">All</a>
<a href=# data-filter=.js-id-Machine-Learning class="btn btn-primary btn-lg">Machine Learning</a>
<a href=# data-filter=.js-id-Bioinformatics class="btn btn-primary btn-lg">Bioinformatics</a>
<a href=# data-filter=.js-id-Software class="btn btn-primary btn-lg">Software</a>
<a href=# data-filter=.js-id-Data-Analysis class="btn btn-primary btn-lg">Data Analysis</a></div></div></div></div><div class="isotope projects-container row js-layout-row"><div class="col-12 isotope-item js-id-Bioinformatics"><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=https://github.com/maxibor/cogsea target=_blank rel=noopener>coGSEA</a></div><a href=https://github.com/maxibor/cogsea target=_blank rel=noopener class=summary-link><div class=article-style>comparative Analysis of Gene Set Enrichment analysis. R and transcriptomics. Also available as a Shiny GUI.</div></a><div class="stream-meta article-metadata"><div><span></span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/maxibor/cogsea target=_blank rel=noopener>Code</a></div></div><div class=ml-3></div></div></div><div class="col-12 isotope-item js-id-Software"><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=https://maxibor.github.io/dixit-diderot target=_blank rel=noopener>Dixit Diderot</a></div><a href=https://maxibor.github.io/dixit-diderot target=_blank rel=noopener class=summary-link><div class=article-style>Recreate the writing of Philosopher Denis Diderot using Markov Chains.</div></a><div class="stream-meta article-metadata"><div><span></span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/maxibor/dixit-diderot target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://maxibor.github.io/dixit-diderot target=_blank rel=noopener>Website</a></div></div><div class=ml-3></div></div></div><div class="col-12 isotope-item js-id-Bioinformatics js-id-Machine-Learning"><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=https://github.com/maxibor/meetdock target=_blank rel=noopener>Meetdock</a></div><a href=https://github.com/maxibor/meetdock target=_blank rel=noopener class=summary-link><div class=article-style>A scoring method for protein-protein docking.</div></a><div class="stream-meta article-metadata"><div><span></span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/maxibor/meetdock target=_blank rel=noopener>Code</a></div></div><div class=ml-3></div></div></div><div class="col-12 isotope-item js-id-Data-Analysis"><div class="media stream-item"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=/project/anacyclus/>Phylogeny of the Anacyclus genus</a></div><a href=/project/anacyclus/ class=summary-link><div class=article-style>Phylogeny of this Moroccan medicinal plant Anacyclus genus using the internal transcribed spacer marker.</div></a><div class="stream-meta article-metadata"><div><span></span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=/oslo_report.pdf target=_blank rel=noopener>PDF</a></div></div><div class=ml-3></div></div></div></div></div></div></div></section><section id=contact class="home-section wg-contact"><div class=home-section-bg></div><div class=container><div class=row><div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start"><h1 class=mb-0>Contact</h1></div><div class="col-12 col-lg-8"><ul class=fa-ul><li><i class="fa-li fas fa-envelope fa-2x" aria-hidden=true></i><span id=person-email><a href=mailto:maxime_borry@eva.mpg.de>maxime_borry@eva.mpg.de</a></span></li><li><i class="fa-li fas fa-map-marker-alt fa-2x" aria-hidden=true></i><a href=https://goo.gl/maps/myb5tikEyKiJYrFj7 target=_blank rel=noopener>Leipzig, Germany</a></li><li><i class="fa-li fab fa-github fa-2x" aria-hidden=true></i><a href=https://github.com/maxibor target=_blank rel=noopener>Github - maxibor</a></li><li><i class="fa-li fab fa-twitter fa-2x" aria-hidden=true></i><a href=https://twitter.com/notmaxib target=_blank rel=noopener>DM Me - @notmaxib</a></li><li><i class="fa-li fas fa-graduation-cap fa-2x" aria-hidden=true></i><a href="https://scholar.google.com/citations?user=oUsxa5sAAAAJ" target=_blank rel=noopener>Google scholar</a></li><li><i class="fa-li fab fa-orcid fa-2x" aria-hidden=true></i><a href=https://orcid.org/0000-0001-9140-7559 target=_blank rel=noopener>ORCID - 0000-0001-9140-7559</a></li></ul></div></div></div></section></div><div class=page-footer><div class=container><footer class=site-footer><p class=powered-by>Â© 2024 Maxime Borry</p><p class=powered-by>Published with <a href="https://wowchemy.com/?utm_campaign=poweredby" target=_blank rel=noopener>Wowchemy</a> â€” the free, <a href=https://github.com/wowchemy/wowchemy-hugo-modules target=_blank rel=noopener>open source</a> website builder that empowers creators.</p></footer></div></div><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code class="tex hljs"></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i>Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i>Download</a><div id=modal-error></div></div></div></div></div><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/instant.page/5.1.0/instantpage.min.js integrity="sha512-1+qUtKoh9XZW7j+6LhRMAyOrgSQKenQ4mluTR+cvxXjP1Z54RxZuzstR/H9kgPXQsVB8IW7DMDFUJpzLjvhGSQ==" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/mermaid/8.8.4/mermaid.min.js integrity="sha512-as1BF4+iHZ3BVO6LLDQ7zrbvTXM+c/1iZ1qII/c3c4L8Rn5tHLpFUtpaEtBNS92f+xGsCzsD7b62XP3XYap6oA==" crossorigin=anonymous title=mermaid></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/highlight.min.js integrity="sha512-TDKKr+IvoqZnPzc3l35hdjpHD0m+b2EC2SrLEgKDRWpxf2rFCxemkgvJ5kfU48ip+Y+m2XVKyOCD85ybtlZDmw==" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/languages/r.min.js></script><script src=/js/bootstrap.bundle.min.6aed84840afc03ab4d5750157f69c120.js></script><script src=/en/js/wowchemy.min.9d61d83fd2ae986071a8cb13e03df865.js></script></body></html>