<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Wowchemy 5.0.0-beta.1 for Hugo"><meta name=description content="Recently, I&rsquo;ve been involved with the AncientMetagenomeDir project. Briefly, with this collaborative community effort, we aimed to regroup in one single repository, all the metadata about every single published ancient DNA metagenomics article, and turn them into FAIR scientific data."><link rel=alternate hreflang=en-us href=https://maximeborry.com/post/interactive-map-howto/><link rel=preconnect href=https://fonts.gstatic.com crossorigin><meta name=theme-color content="#01579b"><script src=/js/mathjax-config.js></script><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css integrity="sha256-FMvZuGapsJLjouA6k7Eo2lusoAX9i0ShlWFG6qt7SLc=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin=anonymous media=print onload="this.media='all'"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/github.min.css crossorigin=anonymous title=hl-light media=print onload="this.media='all'"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/github.min.css crossorigin=anonymous title=hl-dark media=print onload="this.media='all'" disabled><script src=https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.2.2/lazysizes.min.js integrity="sha512-TmDwFLhg3UA4ZG0Eb4MIyT1O1Mb+Oww5kFG0uHqXsdbyZz9DcvYQhKpGgNkamAI6h2lGGZq2X8ftOJvF/XjTUg==" crossorigin=anonymous async></script><script src=https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js integrity crossorigin=anonymous async></script><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Bebas+Neue%7CIBM+Plex+Mono&display=swap"><link rel=stylesheet href=/css/wowchemy.9654d915ac5fd1ad52a0c3965cab325f.css><script async src="https://www.googletagmanager.com/gtag/js?id=UA-77504401-1"></script><script>window.dataLayer=window.dataLayer||[];function gtag(){dataLayer.push(arguments);}
function trackOutboundLink(url,target){gtag('event','click',{'event_category':'outbound','event_label':url,'transport_type':'beacon','event_callback':function(){if(target!=='_blank'){document.location=url;}}});console.debug("Outbound link clicked: "+url);}
function onClickCallback(event){if((event.target.tagName!=='A')||(event.target.host===window.location.host)){return;}
trackOutboundLink(event.target,event.target.getAttribute('target'));}
gtag('js',new Date());gtag('config','UA-77504401-1',{'anonymize_ip':true});document.addEventListener('click',onClickCallback,false);</script><link rel=manifest href=/index.webmanifest><link rel=icon type=image/png href=/images/icon_hu4c4b160219548583b0b35e67c5987186_165157_32x32_fill_lanczos_center_2.png><link rel=apple-touch-icon type=image/png href=/images/icon_hu4c4b160219548583b0b35e67c5987186_165157_192x192_fill_lanczos_center_2.png><link rel=canonical href=https://maximeborry.com/post/interactive-map-howto/><meta property="twitter:card" content="summary_large_image"><meta property="twitter:site" content="@notmaxib"><meta property="twitter:creator" content="@notmaxib"><meta property="og:site_name" content="Maxime Borry"><meta property="og:url" content="https://maximeborry.com/post/interactive-map-howto/"><meta property="og:title" content="Making Interactive maps in Python using GeoJSON and GitHub | Maxime Borry"><meta property="og:description" content="Recently, I&rsquo;ve been involved with the AncientMetagenomeDir project. Briefly, with this collaborative community effort, we aimed to regroup in one single repository, all the metadata about every single published ancient DNA metagenomics article, and turn them into FAIR scientific data."><meta property="og:image" content="https://maximeborry.com/post/interactive-map-howto/featured.png"><meta property="twitter:image" content="https://maximeborry.com/post/interactive-map-howto/featured.png"><meta property="og:locale" content="en-us"><meta property="article:published_time" content="2021-01-12T13:08:30+01:00"><meta property="article:modified_time" content="2021-01-12T13:08:30+01:00"><script type=application/ld+json>{"@context":"https://schema.org","@type":"BlogPosting","mainEntityOfPage":{"@type":"WebPage","@id":"https://maximeborry.com/post/interactive-map-howto/"},"headline":"Making Interactive maps in Python using GeoJSON and GitHub","image":["https://maximeborry.com/post/interactive-map-howto/featured.png"],"datePublished":"2021-01-12T13:08:30+01:00","dateModified":"2021-01-12T13:08:30+01:00","publisher":{"@type":"Organization","name":"Maxime Borry","logo":{"@type":"ImageObject","url":"https://maximeborry.com/images/icon_hu4c4b160219548583b0b35e67c5987186_165157_192x192_fill_lanczos_center_2.png"}},"description":"Recently, I\u0026rsquo;ve been involved with the AncientMetagenomeDir project. Briefly, with this collaborative community effort, we aimed to regroup in one single repository, all the metadata about every single published ancient DNA metagenomics article, and turn them into FAIR scientific data."}</script><script src=https://cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.1.1/cookieconsent.min.js integrity="sha256-5VhCqFam2Cn+yjw61zbBNrbHVJ6SRydPeKopYlngbiQ=" crossorigin=anonymous></script><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.1.1/cookieconsent.min.css integrity="sha256-zQ0LblD/Af8vOppw18+2anxsuaz3pWYyVWi+bTvTH8Q=" crossorigin=anonymous><script>window.addEventListener("load",function(){window.cookieconsent.initialise({"palette":{"popup":{"background":"#01579b","text":"rgb(255, 255, 255)"},"button":{"background":"rgb(255, 255, 255)","text":"#01579b"}},"theme":"classic","content":{"message":"This website uses cookies to ensure you get the best experience on our website.","dismiss":"Got it!","link":"Learn more","href":"https://www.cookiesandyou.com"}})});</script><title>Making Interactive maps in Python using GeoJSON and GitHub | Maxime Borry</title></head><body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents class=page-wrapper><script src=/js/wowchemy-init.min.7264cf0eba3b66951b36da7d2cecf9c5.js></script><aside class=search-modal id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=# aria-label=Close><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control aria-label=Search...></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><div class=page-header><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=/>Maxime Borry</a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=/>Maxime Borry</a></div><div class="navbar-collapse main-menu-item collapse justify-content-end" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=/#hero><span>Home</span></a></li><li class=nav-item><a class=nav-link href=/courses><span>Courses</span></a></li><li class=nav-item><a class=nav-link href=/#posts><span>Blog</span></a></li><li class=nav-item><a class=nav-link href=/#publications><span>Publications</span></a></li><li class=nav-item><a class=nav-link href=/#contact><span>Contact me</span></a></li><li class=nav-item><a class=nav-link href=/maxime_borry_cv.pdf><span>CV</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class=nav-item><a class="nav-link js-search" href=# aria-label=Search><i class="fas fa-search" aria-hidden=true></i></a></li><li class="nav-item dropdown theme-dropdown"><a href=# class=nav-link data-toggle=dropdown aria-haspopup=true aria-label="Display preferences"><i class="fas fa-moon" aria-hidden=true></i></a><div class=dropdown-menu><a href=# class="dropdown-item js-set-theme-light"><span>Light</span></a>
<a href=# class="dropdown-item js-set-theme-dark"><span>Dark</span></a>
<a href=# class="dropdown-item js-set-theme-auto"><span>Automatic</span></a></div></li></ul></div></nav></div><div class=page-body><article class=article><div class="article-container pt-3"><h1>Making Interactive maps in Python using GeoJSON and GitHub</h1><div class=article-metadata><span class=article-date>Jan 12, 2021</span>
<span class=middot-divider></span><span class=article-reading-time>3 min read</span></div></div><div class="article-header article-container featured-image-wrapper mt-4 mb-4" style=max-width:720px;max-height:364px><div style=position:relative><img src=/post/interactive-map-howto/featured_hu735c323bb12acc9c85b3da083adb53a6_434394_720x0_resize_lanczos_2.png alt class=featured-image>
<span class=article-header-caption>Ancient DNA metagenomic sample interactive map</span></div></div><div class=article-container><div class=article-style><p>Recently, I&rsquo;ve been involved with the <a href=https://github.com/spaam-community/ancientMetagenomeDir target=_blank rel=noopener>AncientMetagenomeDir</a> project. Briefly, with this collaborative community effort, we aimed to regroup in one single repository, all the metadata about every single published ancient DNA metagenomics article, and turn them into <a href=https://en.wikipedia.org/wiki/FAIR_data target=_blank rel=noopener>FAIR</a> scientific data.</p><p>We ended with large <code>TSV</code> (table) files regrouping a standardized set of metadata, about each ancient DNA metagenomics sample. Because these are originally archeological data, one of the information that is systematically collected is the geographical location of each sample.</p><p>While static maps were already generated for the <a href=https://doi.org/10.1101/2020.09.02.279570 target=_blank rel=noopener>AncientMegenomeDir publication</a>, we had the opportunity to play with interactive maps for the <a href=https://spaam-community.github.io/AncientMetagenomeDir/#/ target=_blank rel=noopener>website of the project</a>.</p><p>Usually, hosting interactive elements online require some sort of backend framework (like <a href=https://www.streamlit.io/ target=_blank rel=noopener>Streamlit</a> or <a href=https://shiny.rstudio.com/ target=_blank rel=noopener>Shiny</a>) to perform the rendering, however, I wanted to have it as serverless as possible, and this is where the <a href=https://docs.github.com/en/free-pro-team@latest/github/managing-files-in-a-repository/mapping-geojson-files-on-github target=_blank rel=noopener><code>GeoJSON</code> rendering function of GitHub</a> came to the rescue.</p><p>Using GitHub magic, that meant that as long as I would push a <a href=https://geojson.org/ target=_blank rel=noopener><code>GeoJSON</code></a> file on GitHub, it would automatically be rendered as an interactive map, thanks to <a href=https://leafletjs.com/ target=_blank rel=noopener>Leaflet.js</a>.</p><h2 id=from-tsv-to-geojson>From TSV to GeoJSON</h2><p>The question that I was left with: How to go from a <code>TSV</code> table to a <code>GeoJSON</code> file ?
Luckily for me, this is really easy to do thanks to <a href=https://geopandas.org/ target=_blank rel=noopener>GeoPandas</a>.<br>I only needed to make sure that there were a <code>latitude</code> and <code>longitude</code> columns in the <code>TSV</code> files.</p><figure id=figure-geographic-coordinate-system-credit-wikipediahttpsenwikipediaorgwikigeographic_coordinate_system><a data-fancybox href=/post/interactive-map-howto/coordinate-system_hu0fbc7a9d6e3ec1414817c2d1317b72fa_45308_2000x2000_fit_lanczos_2.png data-caption='Geographic coordinate system, credit: <a href="https://en.wikipedia.org/wiki/Geographic_coordinate_system">Wikipedia</a>'><img data-src=/post/interactive-map-howto/coordinate-system_hu0fbc7a9d6e3ec1414817c2d1317b72fa_45308_2000x2000_fit_lanczos_2.png class=lazyload alt width=435 height=340></a><figcaption data-pre=Figure&nbsp; data-post=:&nbsp; class=numbered>Geographic coordinate system, credit: <a href=https://en.wikipedia.org/wiki/Geographic_coordinate_system>Wikipedia</a></figcaption></figure><pre><code class=language-python>import pandas as pd
import geopandas

df = pd.read_csv(&quot;table.tsv&quot;, sep=&quot;\t&quot;)
gdf = geopandas.GeoDataFrame(df, geometry=geopandas.points_from_xy(df.longitude, df.latitude))
gdf.to_file(&quot;output.geo.json&quot;, driver='GeoJSON')
</code></pre><blockquote><p>Instead of pushing to GitHub at every change to check a GeoJSON rendering, you can check a GeoJSON map with the <a href=https://geojson.io/ target=_blank rel=noopener>geojson.io</a> website.</p></blockquote><h2 id=displaying-more-metadata-on-the-map>Displaying more metadata on the map</h2><p>So far, I only used the map to display the latitude and longitude of each sample, but we can actually display more information by changing the color, size, or the shape of each marker(point) for example.</p><p>Refering again the <a href=https://docs.github.com/en/free-pro-team@latest/github/managing-files-in-a-repository/mapping-geojson-files-on-github#styling-features target=_blank rel=noopener>Github documentation</a>, this corresponds to the <code>marker-color</code>, <code>marker-size</code>, or <code>marker-symbol</code>.</p><p>For example, to change the color, we add a <code>marker-color</code> column with the desired color value.</p><table><thead><tr><th><strong>marker-color</strong></th><th>publication_doi</th><th>site_name</th><th>latitude</th><th>longitude</th><th>sample_name</th><th>sample_age</th><th>material</th><th>archive</th><th>archive_accession</th></tr></thead><tbody><tr><td>#009C54</td><td>10.1016/j.quascirev.2017.11.037</td><td>Hässeldala Port</td><td>56.16</td><td>15.01</td><td>HA1.1</td><td>13900</td><td>lake sediment</td><td>ENA</td><td>SRS2040659</td></tr><tr><td>#C22026</td><td>10.3390/geosciences10070270</td><td>Unknown</td><td>53.322</td><td>1.118</td><td>ELF001A_95_S81_ELFM1D1</td><td>6000</td><td>shallow marine sediment</td><td>ENA</td><td>ERS3605424</td></tr></tbody></table><p><strong>Table 1:</strong> Sample data from the AncientMetagenomeDir repository</p><figure id=figure-markers-are-colored-by-property><a data-fancybox href=/post/interactive-map-howto/marker-color_hu5de8302c11b0599c4f8a330d955cd7b3_216765_2000x2000_fit_lanczos_2.png data-caption="Markers are colored by property"><img data-src=/post/interactive-map-howto/marker-color_hu5de8302c11b0599c4f8a330d955cd7b3_216765_2000x2000_fit_lanczos_2.png class=lazyload alt width=632 height=415></a><figcaption data-pre=Figure&nbsp; data-post=:&nbsp; class=numbered>Markers are colored by property</figcaption></figure><p>Here, markers in pink are host-associated single genomes, while markers in light-blue are host-associated metagenomes.</p><h2 id=preventing-overlapping-points>Preventing overlapping points</h2><p>In this dataset, different samples are sometimes coming from a same archeological site. In practice, this means that points will overlap on the map because they share the exact same geographic coordinates.
In Figure 2, for example, you can notice a very dark shadow bellow each marker: that&rsquo;s because there are many overlapping markers present on the spot.</p><p>The problem is that only one marker will be displayed, and the other ones being hidden below.</p><p>To overcome this issue, the little trick is to slightly alter the coordinates of each sample to plot them as distinct points on the map. I did that with random sampling from the normal distribution using <a href=https://numpy.org/doc/stable/reference/random/generated/numpy.random.normal.html target=_blank rel=noopener>Numpy</a> with a very small standard deviation.</p><pre><code class=language-python>import pandas as pd
import geopandas
import numpy as np

df = pd.read_csv(&quot;table.tsv&quot;, sep=&quot;\t&quot;)

sigma = 0.0015

df['new_latitude'] = df['latitude'].apply(lambda x: np.random.normal(x, sigma))
df['new_longitude'] = df['longitude'].apply(lambda x: np.random.normal(x, sigma))

gdf = geopandas.GeoDataFrame(df, geometry=geopandas.points_from_xy(df.new_longitude, df.new_latitude))

gdf.to_file(&quot;output.geo.json&quot;, driver='GeoJSON')
</code></pre><p>Problem solved !</p><figure id=figure-overalapping-markers-are-jittered-around-the-exact-coordinates><a data-fancybox href=/post/interactive-map-howto/marker-jitter_hud42370f9c294ed1624b2767a29b5f442_213929_2000x2000_fit_lanczos_2.png data-caption="Overalapping markers are &amp;lsquo;jittered&amp;rsquo; around the exact coordinates"><img data-src=/post/interactive-map-howto/marker-jitter_hud42370f9c294ed1624b2767a29b5f442_213929_2000x2000_fit_lanczos_2.png class=lazyload alt width=635 height=399></a><figcaption data-pre=Figure&nbsp; data-post=:&nbsp; class=numbered>Overalapping markers are &lsquo;jittered&rsquo; around the exact coordinates</figcaption></figure><h2 id=end-result>End result</h2><p>Finally, thanks to the magic of GitHub GeoJSON rendering, the map can be easily embedded on any web page !</p><script src="https://embed.githubusercontent.com/view/geojson/SPAAM-community/AncientMetagenomeDir/master/assets/analysis/live/ancientmetagenomedir.geo.json?height=400&width=800"></script><p>This is an interactive map, you can click on a marker to look at the details.</p></div><div class=share-box aria-hidden=true><ul class=share><li><a href="https://twitter.com/intent/tweet?url=https://maximeborry.com/post/interactive-map-howto/&text=Making%20Interactive%20maps%20in%20Python%20using%20GeoJSON%20and%20GitHub" target=_blank rel=noopener class=share-btn-twitter><i class="fab fa-twitter"></i></a></li><li><a href="https://www.facebook.com/sharer.php?u=https://maximeborry.com/post/interactive-map-howto/&t=Making%20Interactive%20maps%20in%20Python%20using%20GeoJSON%20and%20GitHub" target=_blank rel=noopener class=share-btn-facebook><i class="fab fa-facebook"></i></a></li><li><a href="mailto:?subject=Making%20Interactive%20maps%20in%20Python%20using%20GeoJSON%20and%20GitHub&body=https://maximeborry.com/post/interactive-map-howto/" target=_blank rel=noopener class=share-btn-email><i class="fas fa-envelope"></i></a></li><li><a href="https://www.linkedin.com/shareArticle?url=https://maximeborry.com/post/interactive-map-howto/&title=Making%20Interactive%20maps%20in%20Python%20using%20GeoJSON%20and%20GitHub" target=_blank rel=noopener class=share-btn-linkedin><i class="fab fa-linkedin-in"></i></a></li><li><a href="whatsapp://send?text=Making%20Interactive%20maps%20in%20Python%20using%20GeoJSON%20and%20GitHub%20https://maximeborry.com/post/interactive-map-howto/" target=_blank rel=noopener class=share-btn-whatsapp><i class="fab fa-whatsapp"></i></a></li><li><a href="https://service.weibo.com/share/share.php?url=https://maximeborry.com/post/interactive-map-howto/&title=Making%20Interactive%20maps%20in%20Python%20using%20GeoJSON%20and%20GitHub" target=_blank rel=noopener class=share-btn-weibo><i class="fab fa-weibo"></i></a></li></ul></div><div class="media author-card content-widget-hr"><a href=https://maximeborry.com/><img class="avatar mr-3 avatar-circle" src=/authors/maxime/avatar_hu4dc3c23d5a8c195732bbca11d7ce61be_114670_270x270_fill_lanczos_center_2.png alt></a><div class=media-body><h5 class=card-title><a href=https://maximeborry.com/></a></h5><h6 class=card-subtitle>Maxime Borry</h6><p class=card-text>Bioinformatician - PhD student at the Max Planck Institute for the Science of Human History</p><ul class=network-icon aria-hidden=true><li><a href=https://github.com/maxibor target=_blank rel=noopener><i class="fab fa-github"></i></a></li><li><a href=https://linkedin.com/in/maximeborry/ target=_blank rel=noopener><i class="fab fa-linkedin"></i></a></li><li><a href=/maxime_borry_cv.pdf><i class="ai ai-cv"></i></a></li><li><a href=/#contact><i class="fas fa-envelope"></i></a></li><li><a href=https://twitter.com/notmaxib target=_blank rel=noopener><i class="fab fa-twitter"></i></a></li></ul></div></div></div></article></div><div class=page-footer><div class=container><footer class=site-footer><p class=powered-by>© 2021 Maxime Borry</p><p class=powered-by>Published with
<a href="https://wowchemy.com/?utm_campaign=poweredby" target=_blank rel=noopener>Wowchemy</a> —
the free, <a href=https://github.com/wowchemy/wowchemy-hugo-modules target=_blank rel=noopener>open source</a> website builder that empowers creators.</p></footer></div></div><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code class="tex hljs"></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i>Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i>Download</a><div id=modal-error></div></div></div></div></div><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/instant.page/5.1.0/instantpage.min.js integrity="sha512-1+qUtKoh9XZW7j+6LhRMAyOrgSQKenQ4mluTR+cvxXjP1Z54RxZuzstR/H9kgPXQsVB8IW7DMDFUJpzLjvhGSQ==" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/mermaid/8.8.0/mermaid.min.js integrity="sha512-ja+hSBi4JDtjSqc4LTBsSwuBT3tdZ3oKYKd07lTVYmCnTCor56AnRql00ssqnTOR9Ss4gOP/ROGB3SfcJnZkeg==" crossorigin=anonymous title=mermaid></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/highlight.min.js integrity="sha512-TDKKr+IvoqZnPzc3l35hdjpHD0m+b2EC2SrLEgKDRWpxf2rFCxemkgvJ5kfU48ip+Y+m2XVKyOCD85ybtlZDmw==" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/languages/r.min.js></script><script>const code_highlighting=true;</script><script>const search_config={"indexURI":"/index.json","minLength":1,"threshold":0.3};const i18n={"no_results":"No results found","placeholder":"Search...","results":"results found"};const content_type={'post':"Posts",'project':"Projects",'publication':"Publications",'event':"Events",'slides':"Slides"};</script><script id=search-hit-fuse-template type=text/x-template>
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script><script src=https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin=anonymous></script><script src=/js/wowchemy.min.aeecf26c1eff6b9588dbafc1a9f3953c.js></script></body></html>