<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Source Themes Academic 4.8.0"><meta name=author content="Maxime Borry"><meta name=description content="Shotgun metagenomics applied to archaeological feces (paleofeces) can bring new insights into the composition and functions of human and animal gut microbiota from the past. However, paleofeces often undergo physical distortions in archaeological sediments, making their source species difficult to identify on the basis of fecal morphology or microscopic features alone. Here we present a reproducible and scalable pipeline using both host and microbial DNA to infer the host source of fecal material. We apply this pipeline to newly sequenced archaeological specimens and show that we are able to distinguish morphologically similar human and canine paleofeces, as well as non-fecal sediments, from a range of archaeological contexts."><link rel=alternate hreflang=en-us href=/publication/coproid/><meta name=theme-color content="#01579b"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-1/css/all.min.css integrity="sha256-4w9DunooKSr3MFXHXWyFER38WmPdm361bQS/2KUWZbU=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/styles/github.min.css crossorigin=anonymous title=hl-light disabled><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/styles/dracula.min.css crossorigin=anonymous title=hl-dark><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Bebas+Neue%7CIBM+Plex+Mono&display=swap"><link rel=stylesheet href=/css/academic.css><script async src="https://www.googletagmanager.com/gtag/js?id=UA-77504401-1"></script><script>window.dataLayer=window.dataLayer||[];function gtag(){dataLayer.push(arguments);}
function trackOutboundLink(url,target){gtag('event','click',{'event_category':'outbound','event_label':url,'transport_type':'beacon','event_callback':function(){if(target!=='_blank'){document.location=url;}}});console.debug("Outbound link clicked: "+url);}
function onClickCallback(event){if((event.target.tagName!=='A')||(event.target.host===window.location.host)){return;}
trackOutboundLink(event.target,event.target.getAttribute('target'));}
gtag('js',new Date());gtag('config','UA-77504401-1',{'anonymize_ip':true});document.addEventListener('click',onClickCallback,false);</script><link rel=manifest href=/index.webmanifest><link rel=canonical href=/publication/coproid/><meta property="twitter:card" content="summary_large_image"><meta property="twitter:site" content="@maxibor"><meta property="twitter:creator" content="@maxibor"><meta property="og:site_name" content="Maxime Borry"><meta property="og:url" content="/publication/coproid/"><meta property="og:title" content="CoproID predicts the source of coprolites and paleofeces using microbiome composition and host DNA content | Maxime Borry"><meta property="og:description" content="Shotgun metagenomics applied to archaeological feces (paleofeces) can bring new insights into the composition and functions of human and animal gut microbiota from the past. However, paleofeces often undergo physical distortions in archaeological sediments, making their source species difficult to identify on the basis of fecal morphology or microscopic features alone. Here we present a reproducible and scalable pipeline using both host and microbial DNA to infer the host source of fecal material. We apply this pipeline to newly sequenced archaeological specimens and show that we are able to distinguish morphologically similar human and canine paleofeces, as well as non-fecal sediments, from a range of archaeological contexts."><meta property="og:image" content="/img/bikepacking.jpeg"><meta property="twitter:image" content="/img/bikepacking.jpeg"><meta property="og:locale" content="en-us"><meta property="article:published_time" content="2020-01-03T18:47:43+01:00"><meta property="article:modified_time" content="2020-01-03T18:47:43+01:00"><script type=application/ld+json>{"@context":"https://schema.org","@type":"Article","mainEntityOfPage":{"@type":"WebPage","@id":"/publication/coproid/"},"headline":"CoproID predicts the source of coprolites and paleofeces using microbiome composition and host DNA content","datePublished":"2020-01-03T18:47:43+01:00","dateModified":"2020-01-03T18:47:43+01:00","author":{"@type":"Person","name":"Maxime Borry​"},"publisher":{"@type":"Organization","name":"Maxime Borry","logo":{"@type":"ImageObject","url":"/images/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_192x192_fill_lanczos_center_2.png"}},"description":"Shotgun metagenomics applied to archaeological feces (paleofeces) can bring new insights into the composition and functions of human and animal gut microbiota from the past. However, paleofeces often undergo physical distortions in archaeological sediments, making their source species difficult to identify on the basis of fecal morphology or microscopic features alone. Here we present a reproducible and scalable pipeline using both host and microbial DNA to infer the host source of fecal material. We apply this pipeline to newly sequenced archaeological specimens and show that we are able to distinguish morphologically similar human and canine paleofeces, as well as non-fecal sediments, from a range of archaeological contexts."}</script><script src=https://cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.1.1/cookieconsent.min.js integrity="sha256-5VhCqFam2Cn+yjw61zbBNrbHVJ6SRydPeKopYlngbiQ=" crossorigin=anonymous></script><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.1.1/cookieconsent.min.css integrity="sha256-zQ0LblD/Af8vOppw18+2anxsuaz3pWYyVWi+bTvTH8Q=" crossorigin=anonymous><script>window.addEventListener("load",function(){window.cookieconsent.initialise({"palette":{"popup":{"background":"#01579b","text":"#fff"},"button":{"background":"#fff","text":"#01579b"}},"theme":"classic","content":{"message":"This website uses cookies to ensure you get the best experience on our website.","dismiss":"Got it!","link":"Learn more","href":"/tags/privacy/"}})});</script><title>CoproID predicts the source of coprolites and paleofeces using microbiome composition and host DNA content | Maxime Borry</title></head><body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents class=dark><aside class=search-results id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=#><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-collapse main-menu-item collapse justify-content-end" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=/#hero><span>Home</span></a></li><li class=nav-item><a class=nav-link href=/#projects><span>Projects</span></a></li><li class=nav-item><a class=nav-link href=/#contact><span>Contact me</span></a></li><li class=nav-item><a class=nav-link href=/maxime_borry_cv.pdf><span>CV</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class=nav-item><a class="nav-link js-search" href=#><i class="fas fa-search" aria-hidden=true></i></a></li><li class=nav-item><a class="nav-link js-dark-toggle" href=#><i class="fas fa-moon" aria-hidden=true></i></a></li></ul></div></nav><div class=pub><div class="article-container pt-3"><h1>CoproID predicts the source of coprolites and paleofeces using microbiome composition and host DNA content</h1><div class=article-metadata><div><span><a href=/authors/maxime-borry/>Maxime Borry​</a></span>, <span><a href=/authors/bryan-cordova/>Bryan Cordova</a></span>, <span><a href=/authors/angela-perri/>Angela Perri</a></span>, <span><a href=/authors/marsha-wibowo/>Marsha Wibowo</a></span>, <span><a href=/authors/tanvi-prasad-honap/>Tanvi Prasad Honap</a></span>, <span><a href=/authors/jada-ko/>Jada Ko</a></span>, <span><a href=/authors/jie-yu/>Jie Yu</a></span>, <span><a href=/authors/kate-britton/>Kate Britton</a></span>, <span><a href=/authors/linus-girdland-flink/>Linus Girdland-Flink</a></span>, <span><a href=/authors/robert-c.-power/>Robert C. Power</a></span>, <span><a href=/authors/ingelise-stuijts/>Ingelise Stuijts</a></span>, <span><a href=/authors/domingo-c.-salazar-garcia/>Domingo C. Salazar-García</a></span>, <span><a href=/authors/courtney-hofman/>Courtney Hofman</a></span>, <span><a href=/authors/richard-hagan/>Richard Hagan</a></span>, <span><a href=/authors/therese-samdapawinde-kagone/>Thérèse Samdapawindé Kagoné</a></span>, <span><a href=/authors/nicolas-meda/>Nicolas Meda</a></span>, <span><a href=/authors/helene-carabin/>Helene Carabin</a></span>, <span><a href=/authors/david-jacobson/>David Jacobson</a></span>, <span><a href=/authors/karl-reinhard/>Karl Reinhard</a></span>, <span><a href=/authors/cecil-lewis/>Cecil Lewis</a></span>, <span><a href=/authors/aleksandar-kostic/>Aleksandar Kostic</a></span>, <span><a href=/authors/choongwon-jeong/>Choongwon Jeong</a></span>, <span><a href=/authors/alexander-herbig/>Alexander Herbig</a></span>, <span><a href=/authors/alexander-hubner/>Alexander Hübner</a></span>, <span><a href=/authors/christina-warinner/>Christina Warinner</a></span></div><span class=article-date>January 2020</span>
<span class=middot-divider></span><span class=article-categories><i class="fas fa-folder mr-1"></i><a href=/categories/metagenomics/>Metagenomics</a>, <a href=/categories/ancient-dna/>ancient DNA</a></span></div><div class="btn-links mb-3"><a class="btn btn-outline-primary my-1 mr-1" href=https://peerj.com/articles/9001.pdf target=_blank rel=noopener>PDF</a>
<a class="btn btn-outline-primary my-1 mr-1" href=https://github.com/nf-core/coproid/ target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary my-1 mr-1" href=https://doi.org/10.7717/peerj.9001 target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary my-1 mr-1" href=https://peerj.com/articles/9001.bib target=_blank rel=noopener>Bibtex</a></div></div><div class=article-container><h3>Abstract</h3><p class=pub-abstract>Shotgun metagenomics applied to archaeological feces (paleofeces) can bring new insights into the composition and functions of human and animal gut microbiota from the past. However, paleofeces often undergo physical distortions in archaeological sediments, making their source species difficult to identify on the basis of fecal morphology or microscopic features alone. Here we present a reproducible and scalable pipeline using both host and microbial DNA to infer the host source of fecal material. We apply this pipeline to newly sequenced archaeological specimens and show that we are able to distinguish morphologically similar human and canine paleofeces, as well as non-fecal sediments, from a range of archaeological contexts.</p><div class=row><div class=col-md-1></div><div class=col-md-10><div class=row><div class="col-12 col-md-3 pub-row-heading">Type</div><div class="col-12 col-md-9"><a href=/publication/#2>Journal article</a></div></div></div><div class=col-md-1></div></div><div class="d-md-none space-below"></div><div class=space-below></div><div class=article-style></div><div class=article-tags><a class="badge badge-light" href=/tags/coprolite/>Coprolite</a>
<a class="badge badge-light" href=/tags/paleofeces/>Paleofeces</a>
<a class="badge badge-light" href=/tags/microbiome/>Microbiome</a>
<a class="badge badge-light" href=/tags/endogenous-dna/>Endogenous DNA</a>
<a class="badge badge-light" href=/tags/archeology/>Archeology</a>
<a class="badge badge-light" href=/tags/machine-learning/>Machine learning</a>
<a class="badge badge-light" href=/tags/nextflow/>Nextflow</a>
<a class="badge badge-light" href=/tags/gut/>Gut</a>
<a class="badge badge-light" href=/tags/human/>Human</a>
<a class="badge badge-light" href=/tags/dog/>Dog</a></div><div class=share-box aria-hidden=true><ul class=share><li><a href="https://twitter.com/intent/tweet?url=/publication/coproid/&text=CoproID%20predicts%20the%20source%20of%20coprolites%20and%20paleofeces%20using%20microbiome%20composition%20and%20host%20DNA%20content" target=_blank rel=noopener class=share-btn-twitter><i class="fab fa-twitter"></i></a></li><li><a href="https://www.facebook.com/sharer.php?u=/publication/coproid/&t=CoproID%20predicts%20the%20source%20of%20coprolites%20and%20paleofeces%20using%20microbiome%20composition%20and%20host%20DNA%20content" target=_blank rel=noopener class=share-btn-facebook><i class="fab fa-facebook"></i></a></li><li><a href="mailto:?subject=CoproID%20predicts%20the%20source%20of%20coprolites%20and%20paleofeces%20using%20microbiome%20composition%20and%20host%20DNA%20content&body=/publication/coproid/" target=_blank rel=noopener class=share-btn-email><i class="fas fa-envelope"></i></a></li><li><a href="https://www.linkedin.com/shareArticle?url=/publication/coproid/&title=CoproID%20predicts%20the%20source%20of%20coprolites%20and%20paleofeces%20using%20microbiome%20composition%20and%20host%20DNA%20content" target=_blank rel=noopener class=share-btn-linkedin><i class="fab fa-linkedin-in"></i></a></li><li><a href="https://web.whatsapp.com/send?text=CoproID%20predicts%20the%20source%20of%20coprolites%20and%20paleofeces%20using%20microbiome%20composition%20and%20host%20DNA%20content%20/publication/coproid/" target=_blank rel=noopener class=share-btn-whatsapp><i class="fab fa-whatsapp"></i></a></li><li><a href="https://reddit.com/submit?url=/publication/coproid/&title=CoproID%20predicts%20the%20source%20of%20coprolites%20and%20paleofeces%20using%20microbiome%20composition%20and%20host%20DNA%20content" target=_blank rel=noopener class=share-btn-reddit><i class="fab fa-reddit-alien"></i></a></li></ul></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/maxime-borry/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/bryan-cordova/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/angela-perri/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/marsha-wibowo/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/tanvi-prasad-honap/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/jada-ko/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/jie-yu/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/kate-britton/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/linus-girdland-flink/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/robert-c.-power/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/ingelise-stuijts/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/domingo-c.-salazar-garcia/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/courtney-hofman/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/richard-hagan/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/therese-samdapawinde-kagone/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/nicolas-meda/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/helene-carabin/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/david-jacobson/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/karl-reinhard/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/cecil-lewis/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/aleksandar-kostic/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/choongwon-jeong/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/alexander-herbig/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/alexander-hubner/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/christina-warinner/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div></div></div><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/highlight.min.js integrity="sha256-eOgo0OtLL4cdq7RdwRUiGKLX9XsIJ7nGhWEKbohmVAQ=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/languages/r.min.js></script><script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDVSrig3Qjjp4h0hnhHF7QKd2OeZqNBJkQ"></script><script src=https://cdnjs.cloudflare.com/ajax/libs/gmaps.js/0.4.25/gmaps.min.js integrity="sha256-7vjlAeb8OaTrCXZkCNun9djzuB2owUsaO72kXaFDBJs=" crossorigin=anonymous></script><script>const code_highlighting=true;</script><script>const isSiteThemeDark=true;</script><script>const search_config={"indexURI":"/index.json","minLength":1,"threshold":0.3};const i18n={"no_results":"No results found","placeholder":"Search...","results":"results found"};const content_type={'post':"Posts",'project':"Projects",'publication':"Publications",'talk':"Talks"};</script><script id=search-hit-fuse-template type=text/x-template>
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script><script src=https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin=anonymous></script><script src=/js/academic.min.c816d323c3a55093dae0829b44ea1ca8.js></script><div class=container><footer class=site-footer><p class=powered-by><a href=/tags/privacy/>privacy</a></p><p class=powered-by>© 2020 Maxime Borry. This website is adapted from the work of <a href=https://github.com/dedeswim/hugo-personal-website>Edoardo Debenedetti</a>, licensed under a <a href=http://creativecommons.org/licenses/by-sa/4.0/>Creative Commons Attribution-ShareAlike 4.0 International License</a> &#183;
Powered by the
<a href=https://sourcethemes.com/academic/ target=_blank rel=noopener>Academic theme</a> for
<a href=https://gohugo.io target=_blank rel=noopener>Hugo</a></a>.
<span class=float-right aria-hidden=true><a href=# class=back-to-top><span class=button_icon><i class="fas fa-chevron-up fa-2x"></i></span></a></span></p></footer></div><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code class="tex hljs"></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i>Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i>Download</a><div id=modal-error></div></div></div></div></div></body></html>