<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Wowchemy 5.2.0 for Hugo"><meta name=description content="How to distinguish between true and false positive assignations with KrakenUniq"><link rel=alternate hreflang=en-us href=https://maximeborry.com/post/kraken-uniq/><link rel=preconnect href=https://fonts.gstatic.com crossorigin><meta name=theme-color content="#01579b"><script src=/js/mathjax-config.js></script><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css integrity="sha256-FMvZuGapsJLjouA6k7Eo2lusoAX9i0ShlWFG6qt7SLc=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/github.min.css crossorigin=anonymous title=hl-light media=print onload="this.media='all'"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/github.min.css crossorigin=anonymous title=hl-dark media=print onload="this.media='all'" disabled><script src=https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js integrity crossorigin=anonymous async></script><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Bebas+Neue%7CIBM+Plex+Mono&display=swap"><link rel=stylesheet href=/css/wowchemy.a2cc92da6d94187535714538e195ecb0.css><script async src="https://www.googletagmanager.com/gtag/js?id=UA-77504401-1"></script><script>window.dataLayer=window.dataLayer||[];function gtag(){dataLayer.push(arguments);}
function trackOutboundLink(url,target){gtag('event','click',{'event_category':'outbound','event_label':url,'transport_type':'beacon','event_callback':function(){if(target!=='_blank'){document.location=url;}}});console.debug("Outbound link clicked: "+url);}
function onClickCallback(event){if((event.target.tagName!=='A')||(event.target.host===window.location.host)){return;}
trackOutboundLink(event.target,event.target.getAttribute('target'));}
gtag('js',new Date());gtag('config','UA-77504401-1',{'anonymize_ip':true});gtag('set',{'cookie_flags':'SameSite=None;Secure'});document.addEventListener('click',onClickCallback,false);</script><link rel=manifest href=/index.webmanifest><link rel=icon type=image/png href=/media/icon_hu4c4b160219548583b0b35e67c5987186_165157_32x32_fill_lanczos_center_2.png><link rel=apple-touch-icon type=image/png href=/media/icon_hu4c4b160219548583b0b35e67c5987186_165157_180x180_fill_lanczos_center_2.png><link rel=canonical href=https://maximeborry.com/post/kraken-uniq/><meta property="twitter:card" content="summary_large_image"><meta property="twitter:site" content="@notmaxib"><meta property="twitter:creator" content="@notmaxib"><meta property="og:site_name" content="Maxime Borry"><meta property="og:url" content="https://maximeborry.com/post/kraken-uniq/"><meta property="og:title" content="A new E-score for KrakenUniq | Maxime Borry"><meta property="og:description" content="How to distinguish between true and false positive assignations with KrakenUniq"><meta property="og:image" content="https://maximeborry.com/post/kraken-uniq/featured.png"><meta property="twitter:image" content="https://maximeborry.com/post/kraken-uniq/featured.png"><meta property="og:locale" content="en-us"><meta property="article:published_time" content="2022-05-10T13:38:45+02:00"><meta property="article:modified_time" content="2022-05-10T13:38:45+02:00"><script type=application/ld+json>{"@context":"https://schema.org","@type":"BlogPosting","mainEntityOfPage":{"@type":"WebPage","@id":"https://maximeborry.com/post/kraken-uniq/"},"headline":"A new E-score for KrakenUniq","image":["https://maximeborry.com/post/kraken-uniq/featured.png"],"datePublished":"2022-05-10T13:38:45+02:00","dateModified":"2022-05-10T13:38:45+02:00","publisher":{"@type":"Organization","name":"Maxime Borry","logo":{"@type":"ImageObject","url":"https://maximeborry.com/media/icon_hu4c4b160219548583b0b35e67c5987186_165157_192x192_fill_lanczos_center_2.png"}},"description":"How to distinguish between true and false positive assignations with KrakenUniq"}</script><script src=https://cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.1.1/cookieconsent.min.js integrity="sha256-5VhCqFam2Cn+yjw61zbBNrbHVJ6SRydPeKopYlngbiQ=" crossorigin=anonymous></script><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.1.1/cookieconsent.min.css integrity="sha256-zQ0LblD/Af8vOppw18+2anxsuaz3pWYyVWi+bTvTH8Q=" crossorigin=anonymous><script>window.addEventListener("load",function(){window.cookieconsent.initialise({"palette":{"popup":{"background":"#01579b","text":"rgb(255, 255, 255)"},"button":{"background":"rgb(255, 255, 255)","text":"#01579b"}},"theme":"classic","content":{"message":"This website uses cookies to ensure you get the best experience on our website.","dismiss":"Got it!","link":"Learn more","href":"https://www.cookiesandyou.com"}})});</script><title>A new E-score for KrakenUniq | Maxime Borry</title></head><body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents class=page-wrapper data-wc-page-id=ac0d5f85e58423f42c7c80c691ba65b7><script src=/js/wowchemy-init.min.5f3407a3a06f15b985245df562932996.js></script><div class=page-header><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container-xl><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=/>Maxime Borry</a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar-content aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=/>Maxime Borry</a></div><div class="navbar-collapse main-menu-item collapse justify-content-end" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=/#hero><span>Home</span></a></li><li class=nav-item><a class=nav-link href=/#publications><span>Publications</span></a></li><li class=nav-item><a class=nav-link href=/courses><span>Courses</span></a></li><li class=nav-item><a class=nav-link href=/#posts><span>Blog</span></a></li><li class=nav-item><a class=nav-link href=/#contact><span>Contact me</span></a></li><li class=nav-item><a class=nav-link href=/maxime_borry_cv.pdf><span>CV</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class="nav-item dropdown theme-dropdown"><a href=# class=nav-link data-toggle=dropdown aria-haspopup=true aria-label="Display preferences"><i class="fas fa-moon" aria-hidden=true></i></a><div class=dropdown-menu><a href=# class="dropdown-item js-set-theme-light"><span>Light</span></a>
<a href=# class="dropdown-item js-set-theme-dark"><span>Dark</span></a>
<a href=# class="dropdown-item js-set-theme-auto"><span>Automatic</span></a></div></li></ul></div></nav></div><div class=page-body><article class=article><div class="article-container pt-3"><h1>A new E-score for KrakenUniq</h1><p class=page-subtitle>using the double-exponential function</p><div class=article-metadata><div><span></span></div><span class=article-date>May 10, 2022</span>
<span class=middot-divider></span><span class=article-reading-time>7 min read</span></div></div><div class="article-header article-container featured-image-wrapper mt-4 mb-4" style=max-width:720px;max-height:479px><div style=position:relative><img src=/post/kraken-uniq/featured_hu6bdd134084c2c2ea375ebf18fabcb0c5_21973_720x0_resize_lanczos_2.png alt class=featured-image></div></div><div class=article-container><div class=article-style><h2 id=introduction>Introduction</h2><p>The task of <a href=https://www.sevenbridges.com/taxonomic-profiling-of-metagenomics-samples/ target=_blank rel=noopener>taxonomic profiling</a> consists of answering the question <em>&ldquo;Who is there ?"</em> in a metagenomic sample.
However, in practice, it turns out to be quite a challenging assignment to tackle, and many solutions have been proposed to address it (for a comparative benchmark, see <a href=https://doi.org/10.1038/s41592-022-01431-4 target=_blank rel=noopener>Meyer et al. 2022</a>).</p><p>One of the central aspect of taxonomic profiling is to distinguishing true positive (taxon actually present in a sample) from false positive (taxon not actually present in sample) taxonomic assignments.</p><p>Today, I want to focus on one particular taxonomic profiler: <a href=https://github.com/fbreitwieser/krakenuniq target=_blank rel=noopener>KrakenUniq</a> (<a href=https://doi.org/10.1186/s13059-018-1568-0 target=_blank rel=noopener>Breitwieser et al. 2018</a>).</p><p>Based on the popular <a href=https://en.wikipedia.org/wiki/K-mer target=_blank rel=noopener>k-mer</a> based taxonomic classifier <a href=https://github.com/DerrickWood/kraken target=_blank rel=noopener>Kraken</a> (<a href=https://doi.org/10.1186/gb-2014-15-3-r46 target=_blank rel=noopener>Wood et al. 2014</a>), KrakenUniq adds a unique (ðŸ˜‰) feature on top of Kraken, the reporting of unique k-mer counts.</p><p>Let me explain:</p><p>The original Kraken program is based on direct k-mer matching. First Kraken builds a database of all present k-mer in each reference genomes, and then compares it to the k-mers found in the query sequences/sequencing reads (fig 1) . If there is a match, it uses a taxonomic tree and the LCA algorithm to assign each query sequence to a given taxon.</p><p><img src=kraken.png alt><br><strong>Figure 1</strong>: The Kraken sequence classification algorithm. <a href=https://genomebiology.biomedcentral.com/articles/10.1186/gb-2014-15-3-r46/figures/1 target=_blank rel=noopener>original here</a></p><p>While this is already a good performing taxonomic profiling method, it suffers from one major drawback: it can not account for duplicated sequences. For example, when reporting the number of reads belonging to each taxon in a sample, Kraken is not able to distinguish an evenly covered genome (likely a true positive, blue genome fig 2) from a genome suffering from read stacking (likely a false positive, purple genome, fig 2). This uneven coverage, often a false positive assignment, can be the consequence of different reasons, but one of the most common is due to reads matching ultra-conserved regions between taxons from the same clade, the so-called <em>read stacking</em>.</p><p><img src=featured.png alt><br><strong>Figure 2</strong>: Two taxons having the same amount of assigned reads, two different scenarios. The blue genome has an evenly distributed coverage, while the purple genome suffers from read stacking. Unique kmers are highlighted with a black perimeter</p><p>To circumvent this problem, KrakenUniq leverages the <a href=https://en.wikipedia.org/wiki/HyperLogLog target=_blank rel=noopener>HLL algorithm</a> to count the unique/distinct k-mers. In practice, in the example of fig 2, Kraken would have reported a k-mer count of 24 for both the blue and purple genome, while KrakenUniq would have reported a k-mer count of 7 for the blue genome, and k-mer count of 2 for the purple genome.</p><p>On top of that, KrakenUniq keeps track of how many of unique k-mers have been found, out all possible unique k-mers for each taxon, and normalizes it by the taxon genome size, which gives an estimation of the &ldquo;coverage&rdquo;.</p><p>So, to summarize, with KrakenUniq, for each taxon, we now have three different metrics instead of &ldquo;just one&rdquo; for Kraken :</p><ul><li>read count per taxon (which we already had with kraken), we&rsquo;ll refer to it later as $R$</li><li>number of unique k-mers (new with KrakenUniq), we&rsquo;ll refer to it later as $K$</li><li>&ldquo;coverage&rdquo; of the k-mers of the clade in the database, we&rsquo;ll refer to it later as $C$</li></ul><p>Note that I&rsquo;ve always mentioned the &ldquo;coverage&rdquo; between quotes, this is because <a href=https://github.com/fbreitwieser/krakenuniq/blob/2ac22bf7681223efa17ffba221231c7faac9da05/src/taxdb.hpp#L1103 target=_blank rel=noopener>KrakenUniq defines it</a> as $C = \frac{K}{genome\ size}$.<br>Because by definition, the maximum number of k-mers (of length $k$) for a sequence of length $L$ can not exceed $L - k + 1$. This means that in the ideal situation, for example the blue genome of fig 2, the &ldquo;coverage&rdquo; will be at best, close to 1, but never greater than 1.</p><p>To try to make sense of these three metrics, <a href=https://doi.org/10.1186/s13059-021-02580-z target=_blank rel=noopener>Guellil et al. 2022</a> came up with a score $E$ combining them, to discriminate true positive from false positive taxonomic assignments.</p><p>$$E = \frac{K}{R} \times C$$</p><p>Here, using different simulated taxon assignment situations, I propose a tweak to this score, to try to improve its ability to differentiate true positive from false positive taxonomic assignments.</p><h2 id=the-simulations>The simulations</h2><p>First, let&rsquo;s define a few parameters for our simulation, and import necessary libraries</p><pre><code class=language-python>import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns
import pandas as pd

read_length = 45
kmer_length = 35
genome_size = 1000000
</code></pre><p><strong>We&rsquo;ll look at four different possible scenarios</strong><br>For each scenario, we&rsquo;ll simulate 100 different situations</p><h4 id=scenario-a-high-duplication-low-coverage-false-positive>scenario A: high duplication, low coverage (false positive)</h4><ul><li>A lot of reads are duplicated, probably coming from conversed regions, or sequencing artifacts</li><li>low coverage</li><li>There are more reads than unique k-mers</li></ul><pre><code class=language-python>a_reads = np.random.randint(10, 300, 100) # randomly choose the number of reads, between 10 and 300
comp_kmer_a = lambda x: int(x / np.random.randint(2,10, 1))  # randomly choose between 2 and 10 times less k-mers
comp_kmer_a_vec = np.vectorize(comp_kmer_a)
a_kmers = comp_kmer_a_vec(a_reads)
plt.plot(a_kmers/a_reads, label=r&quot;$\frac{k}{R}$&quot;)
plt.plot(a_kmers/genome_size, label=&quot;cov&quot;)
plt.title(&quot;scenario A&quot;)
plt.legend(loc=&quot;upper left&quot;);
</code></pre><p><img src=output_6_0.png alt=png></p><h4 id=scenario-b-low-duplication-low-coverage-true-positive>scenario B: low duplication, low coverage (true positive)</h4><ul><li>Very few reads are duplicated</li><li>low coverage</li><li>more unique k-mers than reads (up to $ku = read_length - kmer_length + 1$ more unique k-mers than reads)</li></ul><pre><code class=language-python>b_reads = np.random.randint(10, 300, 100) # randomly choose the number of reads, between 10 and 300
comp_kmer_b = lambda x: min(genome_size - kmer_length + 1, int(x * np.random.randint(2,(read_length - kmer_length + 1), 1))) # randomly choose up to ku more k-mers than reads
comp_kmer_b_vec = np.vectorize(comp_kmer_b)
b_kmers = comp_kmer_b_vec(b_reads)

plt.plot(b_kmers/b_reads, label=r&quot;$\frac{k}{R}$&quot;)
plt.plot(b_kmers/genome_size, label=&quot;cov&quot;)
plt.title(&quot;scenario B&quot;)
plt.legend(loc=&quot;upper left&quot;);
</code></pre><p><img src=output_8_0.png alt=png></p><h4 id=scenario-c-low-duplication-higher-coverage-true-positive>scenario C: low duplication, higher coverage (true positive)</h4><ul><li>same as scenario B, but with a higher coverage</li></ul><pre><code class=language-python>c_reads = np.random.randint(10000, 100000, 100) # # randomly choose the number of reads, between 10000 and 100000
comp_kmer_c = lambda x: min(genome_size - kmer_length + 1, int(x * np.random.randint(2,(read_length - kmer_length + 1), 1)))
comp_kmer_c_vec = np.vectorize(comp_kmer_c)
c_kmers = comp_kmer_c_vec(c_reads)
plt.plot(c_kmers/c_reads, label=r&quot;$\frac{k}{R}$&quot;)
plt.plot(c_kmers/genome_size, label=&quot;cov&quot;)
plt.title(&quot;scenario C&quot;)
plt.legend(loc=&quot;upper left&quot;);
</code></pre><p><img src=output_10_0.png alt=png></p><h4 id=scenario-d-low-duplication-high-coverage>scenario D: low duplication, high coverage</h4><ul><li>more reads than unique kmers (because of the coverage definiton of KrakenUniq)</li><li>high coverage</li></ul><pre><code class=language-python>d_reads = np.random.randint(genome_size*1.1, genome_size*3, 100) # getting between 1.1*genome_size and 3*genome size reads
comp_kmer_d = lambda x: min(genome_size - kmer_length + 1, int(x * np.random.randint(2,(read_length - kmer_length + 1), 1)))
comp_kmer_d_vec = np.vectorize(comp_kmer_d)
d_kmers = comp_kmer_d_vec(d_reads)
plt.plot(d_kmers/d_reads, label=r&quot;$\frac{k}{R}$&quot;)
plt.plot(d_kmers/genome_size, label=&quot;cov&quot;)
plt.title(&quot;scenario D&quot;)
plt.legend(loc=&quot;upper left&quot;);
</code></pre><p><img src=output_12_0.png alt=png></p><h2 id=comparing-e-scores>Comparing E-scores</h2><h3 id=guellil-et-al-e-score>Guellil <em>et al.</em> E-score</h3><p>Now that we have our four different scenarios, let&rsquo;s look at how <a href=https://doi.org/10.1186/s13059-021-02580-z target=_blank rel=noopener>Guellil <em>et al.</em></a> E-score performs at discriminating between true positive (scenario A) and false positive (scenario B, C, and D) taxonomic assignments.</p><pre><code class=language-python>def e_score_guellil(nb_kmer, nb_read, genome_size):
    cov = nb_kmer/genome_size
    
    return((nb_kmer/nb_read) * cov )
e_score_guellil_vec = np.vectorize(e_score_guellil)
</code></pre><pre><code class=language-python>e_score_guellil_a = e_score_guellil_vec(a_kmers, a_reads, genome_size)
e_score_guellil_b = e_score_guellil_vec(b_kmers, b_reads, genome_size)
e_score_guellil_c = e_score_guellil_vec(c_kmers, c_reads, genome_size)
e_score_guellil_d = e_score_guellil_vec(d_kmers, d_reads, genome_size)

plt.plot(e_score_guellil_a, label = &quot;scenario A&quot;)
plt.plot(e_score_guellil_b, label = &quot;scenario B&quot;)
plt.plot(e_score_guellil_c, label = &quot;scenario C&quot;)
plt.plot(e_score_guellil_d, label = &quot;scenario D&quot;)
plt.legend(loc=&quot;upper left&quot;)
plt.title(&quot;Guellil et al. E score&quot;)
plt.ylim(0,0.001);
</code></pre><p><img src=output_17_0.png alt=png></p><p>As we can see, it can be quite tricky in some situations to differentiate a true from a false positive assignment. This is particularly problematic between scenario A and scenario B.</p><h3 id=modified-dexp-e-score>Modified <em>dexp</em> E-score</h3><p>Here I propose a slightly different score to mitigate some of issue.
Instead of defining</p><p>$$E = \frac{K}{R}\times C$$</p><p>I propose to use</p><p>$$E = \frac{K}{R} \times dexp(C)$$</p><p>with $dexp$ being the <a href=https://en.wikipedia.org/wiki/Double_exponential_function target=_blank rel=noopener>double exponential function</a>.</p><p>This way, more emphasis is given to the &ldquo;coverage&rdquo;, which particularly helps in low &ldquo;coverage&rdquo; scenarios, since the double exponential function grows &ldquo;really fast&rdquo; between 0 and 1.</p><pre><code class=language-python>#https://en.wikipedia.org/wiki/Double_exponential_function
def double_exp(x, a=1.3, b=18):
    return(a**(b*x))
double_exp_vec = np.vectorize(double_exp) 
</code></pre><pre><code class=language-python>x = np.arange(0,1,0.01)
y = double_exp_vec(x)
plt.plot(x, y)
plt.title(&quot;Double exponential function&quot;);
</code></pre><p><img src=output_21_0.png alt=png></p><pre><code class=language-python>def e_score_dexp(nb_kmer, nb_read, genome_size):
    cov = nb_kmer/genome_size
    
    return((nb_kmer/nb_read) * double_exp(cov))
e_score_dexp_vec = np.vectorize(e_score_dexp)
</code></pre><p>Now let&rsquo;s look at how this modified E-score performs</p><pre><code class=language-python>e_score_dexp_a = e_score_dexp_vec(a_kmers, a_reads, genome_size)
e_score_dexp_b = e_score_dexp_vec(b_kmers, b_reads, genome_size)
e_score_dexp_c = e_score_dexp_vec(c_kmers, c_reads, genome_size)
e_score_dexp_d = e_score_dexp_vec(d_kmers, d_reads, genome_size)

plt.plot(e_score_dexp_a, label = &quot;case_a&quot;)
plt.plot(e_score_dexp_b, label = &quot;case_b&quot;)
plt.plot(e_score_dexp_c, label = &quot;case_c&quot;)
plt.plot(e_score_dexp_d, label = &quot;case_b&quot;)
plt.legend(loc=&quot;upper left&quot;)
plt.title(&quot;Double-exp E-score&quot;)

plt.ylim(0,4);
</code></pre><p><img src=output_24_0.png alt=png></p><p>We can see that there is a much clearer distinction between true and false positive assignments.
We can also look at the mean difference of all scenarios between true and false positives, for the original, and the new double exp E-score</p><pre><code class=language-python>diff_e_score_guellil = np.mean([e_score_guellil_b - e_score_guellil_a, e_score_guellil_c - e_score_guellil_a, e_score_guellil_d - e_score_guellil_a], axis=0)
diff_e_score_dexp = np.mean([e_score_dexp_b - e_score_dexp_a, e_score_dexp_c - e_score_dexp_a, e_score_dexp_d - e_score_dexp_a], axis=0)
</code></pre><pre><code class=language-python>plt.plot(diff_e_score_guellil, label=&quot;Guellil et al. E-score&quot;)
plt.plot(diff_e_score_dexp, label=&quot;double-exp E-score&quot;)
plt.title(&quot;Difference of E-score between true and false positive\n(the greater the better)&quot;)
plt.legend(loc=&quot;upper left&quot;)
plt.ylim(0,30);
</code></pre><p><img src=output_27_0.png alt=png></p><pre><code class=language-python>df = pd.DataFrame(list(zip(diff_e_score_guellil, diff_e_score_dexp)), columns=['Guellil et al.', 'dexp'])
</code></pre><pre><code class=language-python>sns.boxplot(data=df)
plt.title(&quot;Difference of E-score between true and false positive\n(the greater the better)&quot;);
</code></pre><p><img src=output_29_0.png alt=png></p><pre><code class=language-python>df.mean(axis=0)
</code></pre><pre><code>Guellil et al.     1.012888
dexp              50.104112
dtype: float64
</code></pre><p>We can see that on average between all scenarios, for Guellil <em>et al.</em> E-score, there is only a difference of <code>1</code> point, while for the modified double-exponential E-score, there is an average difference of <code>50</code> points.</p><h3 id=conclusion>Conclusion</h3><p>In my simulations, using a the modified double-exponential E-score could allow for a better distinction between true and false positive taxonomic assignations by KrakenUniq.</p><p><em>The notebook used for this blog post is available</em> <a href=https://gist.github.com/49841c19b4f78c6182b6cce75e9025f2 target=_blank rel=noopener><em>here</em></a></p></div><div class=article-tags><a class="badge badge-light" href=/tag/kraken/>kraken</a>
<a class="badge badge-light" href=/tag/metagenomics/>metagenomics</a>
<a class="badge badge-light" href=/tag/duplication/>duplication</a></div><div class=share-box aria-hidden=true><ul class=share><li><a href="https://twitter.com/intent/tweet?url=https://maximeborry.com/post/kraken-uniq/&text=A%20new%20E-score%20for%20KrakenUniq" target=_blank rel=noopener class=share-btn-twitter><i class="fab fa-twitter"></i></a></li><li><a href="https://www.facebook.com/sharer.php?u=https://maximeborry.com/post/kraken-uniq/&t=A%20new%20E-score%20for%20KrakenUniq" target=_blank rel=noopener class=share-btn-facebook><i class="fab fa-facebook"></i></a></li><li><a href="mailto:?subject=A%20new%20E-score%20for%20KrakenUniq&body=https://maximeborry.com/post/kraken-uniq/" target=_blank rel=noopener class=share-btn-email><i class="fas fa-envelope"></i></a></li><li><a href="https://www.linkedin.com/shareArticle?url=https://maximeborry.com/post/kraken-uniq/&title=A%20new%20E-score%20for%20KrakenUniq" target=_blank rel=noopener class=share-btn-linkedin><i class="fab fa-linkedin-in"></i></a></li><li><a href="whatsapp://send?text=A%20new%20E-score%20for%20KrakenUniq%20https://maximeborry.com/post/kraken-uniq/" target=_blank rel=noopener class=share-btn-whatsapp><i class="fab fa-whatsapp"></i></a></li><li><a href="https://service.weibo.com/share/share.php?url=https://maximeborry.com/post/kraken-uniq/&title=A%20new%20E-score%20for%20KrakenUniq" target=_blank rel=noopener class=share-btn-weibo><i class="fab fa-weibo"></i></a></li></ul></div><div class="media author-card content-widget-hr"><a href=https://maximeborry.com/><img class="avatar mr-3 avatar-circle" src=/authors/maxime/avatar_hu55fedc832821a50ac8f2b09fe94bc985_478322_270x270_fill_lanczos_center_2.png alt></a><div class=media-body><h5 class=card-title><a href=https://maximeborry.com/></a></h5><h6 class=card-subtitle>Maxime Borry, PhD.</h6><p class=card-text>Bioinformatician - Postdoctoral Researcher at the Max Planck Institute for Evolutionary Anthropology</p><ul class=network-icon aria-hidden=true><li><a href=https://github.com/maxibor target=_blank rel=noopener><i class="fab fa-github"></i></a></li><li><a href=https://linkedin.com/in/maximeborry/ target=_blank rel=noopener><i class="fab fa-linkedin"></i></a></li><li><a href=/maxime_borry_cv.pdf><i class="ai ai-cv"></i></a></li><li><a href=/#contact><i class="fas fa-envelope"></i></a></li><li><a href=https://twitter.com/notmaxib target=_blank rel=noopener><i class="fab fa-twitter"></i></a></li></ul></div></div></div></article></div><div class=page-footer><div class=container><footer class=site-footer><p class=powered-by>Â© 2024 Maxime Borry</p><p class=powered-by>Published with <a href="https://wowchemy.com/?utm_campaign=poweredby" target=_blank rel=noopener>Wowchemy</a> â€” the free, <a href=https://github.com/wowchemy/wowchemy-hugo-modules target=_blank rel=noopener>open source</a> website builder that empowers creators.</p></footer></div></div><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code class="tex hljs"></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i>Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i>Download</a><div id=modal-error></div></div></div></div></div><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/instant.page/5.1.0/instantpage.min.js integrity="sha512-1+qUtKoh9XZW7j+6LhRMAyOrgSQKenQ4mluTR+cvxXjP1Z54RxZuzstR/H9kgPXQsVB8IW7DMDFUJpzLjvhGSQ==" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/mermaid/8.8.4/mermaid.min.js integrity="sha512-as1BF4+iHZ3BVO6LLDQ7zrbvTXM+c/1iZ1qII/c3c4L8Rn5tHLpFUtpaEtBNS92f+xGsCzsD7b62XP3XYap6oA==" crossorigin=anonymous title=mermaid></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/highlight.min.js integrity="sha512-TDKKr+IvoqZnPzc3l35hdjpHD0m+b2EC2SrLEgKDRWpxf2rFCxemkgvJ5kfU48ip+Y+m2XVKyOCD85ybtlZDmw==" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/languages/r.min.js></script><script src=/js/bootstrap.bundle.min.6aed84840afc03ab4d5750157f69c120.js></script><script src=/en/js/wowchemy.min.9d61d83fd2ae986071a8cb13e03df865.js></script></body></html>