---
# Documentation: https://sourcethemes.com/academic/docs/managing-content/

title: "CoproID predicts the source of coprolites and paleofeces using microbiome composition and host DNA content"
authors: ["Maxime Borry​", 
          "Bryan Cordova", 
          "Angela Perri", 
          "Marsha Wibowo", 
          "Tanvi Prasad Honap", 
          "Jada Ko", 
          "Jie Yu", 
          "Kate Britton", 
          "Linus Girdland-Flink",
          "Robert C. Power", 
          "Ingelise Stuijts", 
          "Domingo C. Salazar-García", 
          "Courtney Hofman", 
          "Richard Hagan", 
          "Thérèse Samdapawindé Kagoné", 
          "Nicolas Meda", 
          "Helene Carabin", 
          "David Jacobson", 
          "Karl Reinhard", 
          "Cecil Lewis", 
          "Aleksandar Kostic", 
          "Choongwon Jeong", 
          "Alexander Herbig", 
          "Alexander Hübner", 
          "Christina Warinner"]
date: 2020-17-04
doi: "10.7717/peerj.9001"

# Schedule page publish date (NOT publication's date).
publishDate: 2020-01-03T18:47:43+01:00

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: ""
publication_short: ""

abstract: "Shotgun metagenomics applied to archaeological feces (paleofeces) can bring new insights into the composition and functions of human and animal gut microbiota from the past. However, paleofeces often undergo physical distortions in archaeological sediments, making their source species difficult to identify on the basis of fecal morphology or microscopic features alone. Here we present a reproducible and scalable pipeline using both host and microbial DNA to infer the host source of fecal material. We apply this pipeline to newly sequenced archaeological specimens and show that we are able to distinguish morphologically similar human and canine paleofeces, as well as non-fecal sediments, from a range of archaeological contexts."

# Summary. An optional shortened abstract.
summary: ""

tags: ["Coprolite", "Paleofeces", "Microbiome", "Endogenous DNA", "Archeology", "Machine learning", "Nextflow", "Gut", "Human", "Dog"]
categories: ["Metagenomics", "ancient DNA"]
featured: true

# Custom links (optional).
#   Uncomment and edit lines below to show custom links.
links:
- name: Bibtex
  url: "https://peerj.com/articles/9001.bib"
  icon_pack: fab
  icon: 
url_pdf: "https://peerj.com/articles/9001.pdf"
url_code: "https://github.com/nf-core/coproid/"
url_dataset:
url_poster:
url_project:
url_slides:
url_source:
url_video:

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ""
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: ""
---
