---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Reconstruction of ancient microbial genomes from the human gut
subtitle: ''
summary: ''
authors:
- Marsha C. Wibowo
- Zhen Yang
- Maxime Borry
- Alexander Hübner
- Kun D. Huang
- Braden T. Tierney
- Samuel Zimmerman
- Francisco Barajas-Olmos
- Cecilia Contreras-Cubas
- Humberto García-Ortiz
- Angélica Martínez-Hernández
- Jacob M. Luber
- Philipp Kirstahler
- Tre Blohm
- Francis E. Smiley
- Richard Arnold
- Sonia A. Ballal
- Sünje Johanna Pamp
- Julia Russ
- Frank Maixner
- Omar Rota-Stabelli
- Nicola Segata
- Karl Reinhard
- Lorena Orozco
- Christina Warinner
- Meradeth Snow
- Steven LeBlanc
- Aleksandar D. Kostic
tags: []
categories: []
date: '2021-05-01'
lastmod: 2021-05-18T15:53:01+02:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2021-05-18T13:53:01.058822Z'
publication_types:
- '2'
abstract: Loss of gut microbial diversity in industrial populations is associated
  with chronic diseases, underscoring the importance of studying our ancestral gut
  microbiome. However, relatively little is known about the composition of pre-industrial
  gut microbiomes. Here we performed a large-scale de novo assembly of microbial genomes
  from palaeofaeces. From eight authenticated human palaeofaeces samples (1,000–2,000 years
  old) with well-preserved DNA from southwestern USA and Mexico, we reconstructed
  498 medium- and high-quality microbial genomes. Among the 181 genomes with the strongest
  evidence of being ancient and of human gut origin, 39% represent previously undescribed
  species-level genome bins. Tip dating suggests an approximate diversification timeline
  for the key human symbiont Methanobrevibacter smithii. In comparison to 789 present-day
  human gut microbiome samples from eight countries, the palaeofaeces samples are
  more similar to non-industrialized than industrialized human gut microbiomes. Functional
  profiling of the palaeofaeces samples reveals a markedly lower abundance of antibiotic-resistance
  and mucin-degrading genes, as well as enrichment of mobile genetic elements relative
  to industrial gut microbiomes. This study facilitates the discovery and characterization
  of previously undescribed gut microorganisms from ancient microbiomes and the investigation
  of the evolutionary history of the human gut microbiota through genome reconstruction
  from palaeofaeces.
publication: '*Nature*'
url_pdf: https://www.nature.com/articles/s41586-021-03532-0.pdf
doi: 10.1038/s41586-021-03532-0
---
